---
title: "Data"
# author: Yanyu Liang
date: "`r format(Sys.time(), '%d %B, %Y')`"
---

# Intermediate GWAS

```{r, echo=FALSE}
library(pander)
panderOptions('table.split.table', Inf)
data.inter <- data.frame(id = c(), phenotype = c(), num.of.traits = c(), sample.size = c(), num.of.variants = c(), heritability.mean = c(), ancestry = c())
genDataLine <- function(id, phenotype, num.of.traits, sample.size, num.of.variants, ancestry, heritability.mean = NA) {
  data.inter <- data.frame(id = id, phenotype = phenotype, num.of.traits = num.of.traits, sample.size = sample.size, num.of.variants = num.of.variants, heritability.mean = heritability.mean, ancestry = ancestry)
  return(data.inter)
}
id.counter <- 1
```

## Blood cell phenotype

```{r, echo=FALSE}
data.inter.blood <- genDataLine(id = c(id.counter), phenotype = c('blood cell traits'), num.of.traits = c(36), sample.size = c(173480), num.of.variants = c(29.5e6), heritability.mean = c('10%-30%'), ancestry = c('european'))
id.counter <- id.counter + 1
data.inter <- rbind(data.inter, data.inter.blood)
```

* [Data](ftp://ftp.sanger.ac.uk/pub/project/humgen/summary_statistics/human/2017-12-12/)
* [Paper](http://www.cell.com/cell/fulltext/S0092-8674(16)31463-5)
* Meta-data

```{r, echo=FALSE}
pander(data.inter.blood)
```

The phenotype information is as follow (Table S1)
```{r, echo=FALSE}
library(dplyr)
library(kableExtra)
library(knitr)
data <- read.csv('../docs-data/10.1016.j.cell.2016.10.042-mmc1.csv')#rmd/docs-data/10.1016.j.cell.2016.10.042-mmc1.csv')
kable(data, 'html') %>%
  kable_styling() %>%
  scroll_box(width = '1000px', height = '500px')
```

## Circulating metabolites

```{r, echo=FALSE}
data.inter.metabolite <- genDataLine(id = c(id.counter), phenotype = c('circulating metabolites'), num.of.traits = c(123), sample.size = c(24925), num.of.variants = c(39e6), heritability.mean = c(NA), ancestry = c('european'))
id.counter <- id.counter + 1
data.inter <- rbind(data.inter, data.inter.metabolite)
```

* [Data](http://www.computationalmedicine.fi/data#NMR_GWAS)
* [Paper](https://www.nature.com/articles/ncomms11122)
* Meta-data

```{r, echo=FALSE}
pander(data.inter.metabolite)
```

The phenotype information is [here](https://github.com/liangyy/phenotype-epistasis/blob/master/docs-data/ncomms11122-s1-table-s1.pdf) (Table S1)

## Personality

```{r, echo=FALSE}
data.inter.personality <- genDataLine(id = c(id.counter), phenotype = c('personality'), num.of.traits = c(5), sample.size = c(17375), num.of.variants = c(NA), heritability.mean = c('1%-12%'), ancestry = c('european'))
id.counter <- id.counter + 1
data.inter <- rbind(data.inter, data.inter.personality)
```

* [Paper-complete](https://www.nature.com/articles/ng.3736)
* [Paper-summary statistic available](http://www.tweelingenregister.org/fileadmin/user_upload/publicaties/verslaggeving/NTR-publicaties_2012/Moor_MP_2012.pdf)
* [Data](http://www.tweelingenregister.org/GPC/)
* Meta-data

```{r, echo=FALSE}
pander(data.inter.personality)
```

The phenotype information is [here](https://github.com/liangyy/phenotype-epistasis/blob/master/docs-data/ng.3736-S1-table-s2.pdf) (Table S2)

Keep in mind that the sample size is 17375 (63661 for neuroticism and extraversion) even if it is explicitly told in the readme file).

## C-reactive protein

```{r, echo=FALSE}
data.inter.crp <- genDataLine(id = c(id.counter), phenotype = c('C reactive protein'), num.of.traits = c(1), sample.size = c(5000), num.of.variants = c(2.4e7), heritability.mean = c('16%'), ancestry = c('european'))
id.counter <- id.counter + 1
data.inter <- rbind(data.inter, data.inter.crp)
```

* [Data](ftp://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/PrinsBP_28887542_GCST005067)
* [Paper](http://europepmc.org/articles/PMC5591265;jsessionid=5D3CD9A68BA57635985D74A3530E36BF)
* Meta-data

```{r, echo=FALSE}
pander(data.inter.crp)
```

There are many other phenotypes, health-related biomarkers, that the paper has collected, see [here](http://europepmc.org/articles/PMC5591265/table/Tab1/) for a complement list.


## Summary

```{r, echo=FALSE}
pander(data.inter)
```

# Disease Phenotypes

The individual level data is needed here since we need to calculate the predicted intermediate trait from it.

## WTCCC

[WTCCC (2009)](https://www.nature.com/articles/nature05911) has 7 complex diseases:

1. bipolar disorder (BD)
2. coronary artery disease (CAD)
3. Crohnâ€™s disease (CD)
4. hypertension (HT)
5. rheumatoid arthritis (RA)
6. type 1 diabetes (T1D)
7. type 2 diabetes (T2D)

2000 cases each and 3000 shared controls (european ancestry). Controls contains two groups, 1500 from 1958 British Birth Cohort (58C) and 1500 from blood donors.

Posterior mean genotype is available from Nick in BIMBAM format.

The GWAS result of WTCCC 2009 is [here](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2719288/figure/F3/).

## Autism

TODO

# GWAS for PRS Validation

Here, we focus on WTCCC phenotypes.

## Type 2 Diabetes

```{r, echo=FALSE}
id.counter <- 1
data.valid <- data.frame()
data.valid.t2d <- genDataLine(id = c(id.counter), phenotype = c('t2d'), num.of.traits = c(1), sample.size = c(150000), num.of.variants = c(127903), heritability.mean = c('30%-70%'), ancestry = c('european + Pakistani'))
id.counter <- id.counter + 1
data.valid <- rbind(data.valid, data.valid.t2d)
```

* [Data, DIAGRAMv3.2016MAY31](ftp://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/MorrisAP_22885922_GCST005047)
* [Paper, DIAGRAMv3](https://www-nature-com.proxy.uchicago.edu/articles/ng.2383)
* **ATTENTION: WTCCC is included**
* Meta-data

```{r, echo=FALSE}
pander(data.valid.t2d)
```

The sample information is [here](https://github.com/liangyy/phenotype-epistasis/blob/master/docs-data/ng.2383-S1-table-s1.pdf) (Table S1)

## Crohn's Disease

```{r, echo=FALSE}
data.valid.cd <- genDataLine(id = c(id.counter), phenotype = c('crohn\'s disease'), num.of.traits = c(1), sample.size = c(12000), num.of.variants = c(9.5e6), heritability.mean = c(NA), ancestry = c('european'))
id.counter <- id.counter + 1
data.valid <- rbind(data.valid, data.valid.cd)
```

* [Data](ftp://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/deLangeKM_28067908_GCST004132)
* [Paper, Crohn's disease](https://www.nature.com/articles/ng.3760)
* Meta-data

```{r, echo=FALSE}
pander(data.valid.cd)
```

The sample information is as follow (Table S1).

```{r}
data <- read.csv('../docs-data/ng.3760-S2-table-s1.csv')
kable(data, 'html') %>%
  kable_styling() %>%
  scroll_box(width = '1000px', height = '500px')
```

## Summary

```{r, echo=FALSE}
pander(data.valid)
```
