---
title: "Implementation"
# author: Yanyu Liang
date: "`r format(Sys.time(), '%d %B, %Y')`"
---

$$
\newcommand\logit{\text{logit}}  
$$

# Step 1: $(\beta_I, X_T) \rightarrow \hat{Y}_I$

Current available $f: (\beta_I, X_T) \rightarrow \hat{Y}_I$ are `PRSice` and `LDpred` (see [here](computation.html#1_computing_intermediate_phenotype)). The available $X_T$, WTCCC genotype, is in BIMBAM format.

## Prepare $X_T$

Both `PRSice` and `LDpred` take plink binary, so $X_T$ should be converted to plink binary format. It can be obtained by [`fcgene` chapter 9](https://vcru.wisc.edu/simonlab/bioinformatics/programs/fcgene/fcgene-1.0.7.pdf).

> Side notes: It turns out there are several difficulties to use `fcgene`:
>
> 1. `fcgene` can only take genotype distribution file instead of mean genotype file. One way to work around is to generate a "fake" genotype distribution file. Namely, genotype distribution is set to one-hot vector where only the genotype that is closest to the mean genotype is assigned to 1;
> 2. `fcgene` flips the minor and major alleles. So, to work around, we need to flip the order of allele in genotype distribution file (label it as `genotype-dist-flip`). Note that only the order in genotype file matters and SNP info file does not;
> 3. `fcgene` takes an information file for SNPs with MAF, which is not available but since it is not included in the output we can add a fake MAF (say 0.5).

```
# Pseudo-code for this task

## comment 1 & 2
$ mean2dist_genotype --input test.genotype-mean.txt --output test.genotype-dist-flip.txt

## comment 3
$ gen_snpinfo --input test.pos --output test.bimbam.snpinfo

## run fcgene
$ fcgene --wgd test.genotype-dist-flip.txt --pos test.bimbam.snpinfo --oformat plink-bed --out test.bplink

## outcome files
$ ls
test.bplink.bed test.bplink.bim test.bplink.fam

## *.fam output by fcgene needs post-processing to remove extra space at the end of each column
```

## Prepare $\beta_I$

The following format would work well for both methods.

```
chr     pos     ref     alt     reffrq  info    rs       pval    effalt
chr1  1020428 C T 0.85083 0.98732 rs6687776 0.0587  -0.0100048507289348
chr1  1020496 G A 0.85073 0.98751 rs6678318 0.1287  -0.00826075392985992
```

## Pipeline 

Only the approach using Ldpred has been implemented (at [`code/step1/`](https://github.com/liangyy/phenotype-epistasis/tree/master/code/step1)) 

<!-- ## Prepare fake $Y_I$ -->

<!-- `PRSice` (but not `LDpred`) needs $Y_t$ to optimize prediction accuracy by tuning hyper-parameters. As the true phenotype is not available, here we generate a "fake" $Y_t$ instead which won't affect the computation. -->

## Evaluation

Leave it for future.

# Step 2: $Y_T \sim X_T \cdot \hat{Y}_I$

Current we test the following two types of interactions
\begin{align*}
  \logit(\Pr(Y_T|X_T, Y_I)) &= \alpha + \gamma Y_I + \beta_1 X_T + \beta_2 X_T Y_I\\
  \logit(\Pr(Y_T|X_T, Y_I)) &= \alpha + \gamma Y_I + \beta_{11} X_T + \beta_{12} X_T Y_I + \beta_{21} X_T + \beta_{22} X_T Y_I 
\end{align*}
, where the first is additive model and the second is dominant/recessive model. See detail [here](computation.html#model).

## Combine $X_T$ and $Y_T$

```
# change the last column to phenotype (1 unaffected, 2 affected)
$ bash add_phenotype.sh -p 1 -i data1.fam -o data1_phenotyped.fam
$ bash add_phenotype.sh -p 2 -i data2.fam -o data2_phenotyped.fam

# rename *.bim and *.bed files accordingly
$ mv data1.bim data1_phenotyped.bim.fam; mv data1.bed data1_phenotyped.bed
$ mv data2.bim data2_phenotyped.bim.fam; mv data2.bed data2_phenotyped.bed

# merge two data sets
$ plink --bfile data1_phenotyped --bmerge data2_phenotyped --make-bed --out merged_data

# QC
$ plink --bfile merged_data --make-bed --mind 0.05 --maf 0.01 --geno 0.05 --hwe 0.05 --out merged_data_QC
```

## Get PCs of $X_T$

```
Rscript getQC.R --geno merged_data_QC -nPCs 2 --output PCs.merged_data_QC.txt.gz
```

## Determine the direction of effect

### Direction of $X_T$

```
$ plink --bfile merged_data_QC --logistic --out log_assoc_merged_data_QC --covar <(zcat < PCs.merged_data_QC.txt.gz)
$ Rscript extractDirection.R --input log_assoc.merged_data_QC.logistic --output direction_XT.merge_data_QC.txt.gz
```

### Direction of $Y_I$

```
# predict Y_I based on merged_data_QC
$ python ldpred/validate.py SOMETHING --input merged_data_QC --out merged_data_QC_YI  # ignore the prediction accuracy since the phenotype is wrong

# determine the direction
$ Rscript computeDirection.R --input merged_data_QC_YI.txt.gz --output directon_YI.merged_data_QC.txt.gz
```

## Perform interaction test

```
# merge covariates
bash merge_two_covar.sh -a YI.merged_data_QC.txt.gz -b PCs.merged_data_QC.txt.gz -o covars.merged_data_QC.txt.gz

# additive model
$ plink --bfile merged_data_QC --logistic --covar <(zcat < YI.merged_data_QC.txt.gz) --interaction --out interaction_add_YI.merged_data_QC

# dominant/recessive model
## note that the dominant/recessive is respective to reference allele but not the risk one
## in the end, the dominant and recessive will be reassigned according to direction files
$ plink --bfile merged_data_QC --logistic --covar <(zcat < YI.merged_data_QC.txt.gz) --interaction --dominant --out interaction_dom_YI.merged_data_QC
$ plink --bfile merged_data_QC --logistic --covar <(zcat < YI.merged_data_QC.txt.gz) --interaction --recessive --out interaction_rec_YI.merged_data_QC
```
