---
title: "Implementation"
# author: Yanyu Liang
date: "`r format(Sys.time(), '%d %B, %Y')`"
---

# Step 1: $(\beta_I, X_T) \rightarrow \hat{Y}_I$

Current available $f: (\beta_I, X_T) \rightarrow \hat{Y}_I$ are `PRSice` and `LDpred` (see [here](computation.html#1_computing_intermediate_phenotype)). The available $X_T$, WTCCC genotype, is in BIMBAM format.

## Prepare $X_T$

Both `PRSice` and `LDpred` take plink binary, so $X_T$ should be converted to plink binary format. It can be obtained by [`fcgene` chapter 9](https://vcru.wisc.edu/simonlab/bioinformatics/programs/fcgene/fcgene-1.0.7.pdf).

> Side notes: It turns out there are several difficulties to use `fcgene`:
>
> 1. `fcgene` can only take genotype distribution file instead of mean genotype file. One way to work around is to generate a "fake" genotype distribution file. Namely, genotype distribution is set to one-hot vector where only the genotype that is closest to the mean genotype is assigned to 1;
> 2. `fcgene` flips the minor and major alleles. So, to work around, we need to flip the order of allele in genotype distribution file (label it as `genotype-dist-flip`). Note that only the order in genotype file matters and SNP info file does not;
> 3. `fcgene` takes an information file for SNPs with MAF, which is not available but since it is not included in the output we can add a fake MAF (say 0.5).

```
# Pseudo-code for this task

## comment 1 & 2
$ mean2dist_genotype --input test.genotype-mean.txt --output test.genotype-dist-flip.txt

## comment 3
$ gen_snpinfo --input test.pos --output test.bimbam.snpinfo

## run fcgene
$ fcgene --wgd test.genotype-dist-flip.txt --pos test.bimbam.snpinfo --oformat plink-bed --out test.bplink

## outcome files
$ ls
test.bplink.bed test.bplink.bim test.bplink.fam
```

## Prepare $\beta_I$

The following format would work well for both methods.

```
chr     pos     ref     alt     reffrq  info    rs       pval    effalt
chr1  1020428 C T 0.85083 0.98732 rs6687776 0.0587  -0.0100048507289348
chr1  1020496 G A 0.85073 0.98751 rs6678318 0.1287  -0.00826075392985992
```

## Prepare fake $Y_I$

`PRSice` (but not `LDpred`) needs $Y_t$ to optimize prediction accuracy by tuning hyper-parameters. As the true phenotype is not available, here we generate a "fake" $Y_t$ instead which won't affect the computation.

## Evaluation

Compute
