---
title: "Home"
# author: Yanyu Liang
date: "`r format(Sys.time(), '%d %B, %Y')`"
---

# Data

Go to [data](data.html)

## Intermediate GWAS

The candidate phenotypes/traits should be highly inheritable. The intuition is that it may capture one aspect of genetic background in an as informative as possible manner.

Here is a list of publicly available GWAS summary statistics [link](https://www.ebi.ac.uk/gwas/downloads/summary-statistics).

## Disease Phenotypes

1. WTCCC: type 1 diabetes, type 2 diabetes, coronary heart disease, hypertension, bipolar disorder, rheumatoid arthritis and Crohn's disease
2. Autism

## GWAS for PRS Validation

GWAS summary statistics that can be used to benchmark the validation of PRS calculation.

# Computation

Go to [computation](computation.html).

## Computing intermediate phenotype

Methods available:

* P+T
* LDpred

## Testing interaction

Hypothesis testing under linear model.

## Gene-based analysis

The idea is the test whether predicted intermediate trait interacts with predicted gene expression level.

# Implementation

Go to [implementation](implementation.html) for details.

Pipeline overview is as follow.

Let

1. $\beta_I$ be summary statistic of intermediate GWAS
2. $Y_t$ be phenotype of interest with corresponding genotype $X_t$
3. $\hat{Y}_I$ be the predicted intermediate phenotype
4. $I_T$ be interaction of $X_T$ and $\hat{Y}_I$

```{r, echo=FALSE, engine="tikz", fig.ext='pdf', fig.align='center', out.width=150, out.height=300}
\usetikzlibrary{fit,calc,positioning,decorations.pathreplacing,matrix}
\usetikzlibrary{positioning}
\usetikzlibrary{shapes, arrows, calc, positioning}
\begin{tikzpicture}[auto]
  \tikzstyle{node}=[draw, rectangle, text centered, node distance=5em]
  \tikzstyle{arrow}=[draw, ->]
  \node [node] (summary-statistic) {$\beta_I$};
  \node [node, below of=summary-statistic] (intermediate-phenotype) {$\hat{Y}_I$};
  \node [node, below of=intermediate-phenotype] (genotype) {$X_T$};
  \node [node, below of=genotype] (phenotype) {$Y_T$};
  \node [node, right of=intermediate-phenotype] (interaction) {$I_T = \hat{Y}_I \cdot X_T$};
  \node [node, right of=phenotype] (association) {$Y_T \sim I_T$};
  \path [arrow] (summary-statistic) -- (intermediate-phenotype);
  \path [arrow] (genotype) -- (intermediate-phenotype);
  \path [arrow] (intermediate-phenotype) -- (interaction);
  \path [arrow] (genotype) -- (interaction);
  \path [arrow] (phenotype) -- (association);
  \path [arrow] (interaction) -- (association);
\end{tikzpicture}
```

## Obtaining $\hat{Y}_I$

Current available $f: (\beta_I, X_T) \rightarrow \hat{Y}_I$ are `PRSice` and `LDpred` (see [here](computation.html#1_computing_intermediate_phenotype)).

# Results

Traits that have been tested in WTCCC data:

1. Coronary heart disease
2. Crohn's disease
3. Type II diabetes

Go to [results](results.html) for detail.
