---
title: "Results"
# author: Yanyu Liang
date: "`r format(Sys.time(), '%d %B, %Y')`"
---

```{r, echo=FALSE}
library(yaml)
library(data.table)
library(ggplot2)
reportThre <- function(meta) {
  threshold <- meta$`functional-subsetting`$`pval-theshold`
  cat('The threhold used for extracting functionally consistent variant is', threshold, '\n')
}
plotImage <- function(meta) {
  name <- paste0('https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-direction_XT_YI/', meta$`TRUE`, '__', meta$x1, '__', meta$x2, '/direction_yi.png')
  cat(paste0('![](', name, ')'), '\n')
  cat('\n')
}
listReports <- function(meta) {
  cat(paste0('1. Comparison of Beta [here](', 'https://rawgit.com/liangyy/phenotype-epistasis/master/code/step1/output-run_step1/', meta$`TRUE`, '__', meta$x1, '__', meta$x2, '/prs_report.html', ')'), '\n')
  cat(paste0('2. Marginal directions [here](https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-direction_XT_YI/', meta$`TRUE`, '__', meta$x1, '__', meta$x2, '/direction_report.html)'), '\n')
}
listSubset <- function(filename, subset) {
  meta <- yaml.load_file(filename)
  cat('###', subset, '\n', '\n')
  cat(paste0('Interaction test results are [here](https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-run_step2/', meta$`TRUE`, '__', meta$x1, '__', meta$x2, '/inter_report.', subset, '.html)'), '\n')
}
doAll <- function(filename) {
  meta <- yaml.load_file(filename)
  reportThre(meta)
  plotImage(meta)
  listReports(meta)
}
getR2 <- function(x, ld) {
  o <- unlist(ld[ld$SNP_A == x[1] & ld$SNP_B == x[2], 'R2'])
  if(length(o) == 0) return(NA)
  return(o)
}
drawLD <- function(rs.list, filename) {
  nsnp <- length(rs.list)
  meta <- yaml.load_file(filename)
  ld.filename <- paste0('../code/step2/output-run_step2/', meta$`TRUE`, '__', meta$x1, '__', meta$x2, '/sig_subset.ld')
  ld <- fread(ld.filename)
  snp1 <- rep(rs.list, each = nsnp)
  snp2 <- rep(rs.list, nsnp)
  ld.df <- data.frame(snp1 = snp1, snp2 = snp2, r2 = -1)
  temp <- apply(ld.df, 1, getR2, ld = ld)
  ld.df$r2 <- temp
  ld.df$snp1 <- factor(ld.df$snp1, levels = rs.list)
  ld.df$snp2 <- factor(ld.df$snp2, levels = rs.list)
  p <- ggplot(ld.df) + geom_raster(aes(x = snp1, y = snp2, fill = r2)) + scale_fill_gradient(limits = c(0, 1), low = 'red', high = 'black' ) + ggtitle('LD R2 between selected SNPs') + theme(axis.text.x = element_text(angle = 45, hjust = 1))
  print(p)
}
drawGraph <- function(gy, gi, yi, inter, filename='test'){
  tempfilepref <- 'temp-results-plot'
  sink(paste0(tempfilepref, '.tex'))
  t <- cat(paste0('\\', 'documentclass[preview]{standalone}
  \\', 'usepackage{tikz}
  \\', 'begin{document}
  \\', 'usetikzlibrary{fit,calc,positioning,decorations.pathreplacing,matrix}
  \\', 'usetikzlibrary{positioning}
  \\', 'usetikzlibrary{shapes, arrows, calc, positioning}
  \\', 'begin{tikzpicture}[auto]
  \\', 'tikzstyle{node}=[draw, rectangle, text centered, node distance=5em]
  \\', 'tikzstyle{inter}=[draw, circle,fill=black,inner sep=0pt,minimum size=3pt]
  \\', 'tikzstyle{arrow}=[draw, ->]
  \\', 'node [node] (y) {$Y$};
  \\', 'node [inter, below of=y] (inter) {};
  \\', 'node [node, below left of=inter] (g) {$G$};
  \\', 'node [node, below right of=inter] (i) {$I$};
  \\', 'path [arrow] (g) -- node [above,midway] {', gy, '} (y);
  \\', 'path [arrow] (g) -- node [above,midway] {', gi, '} (i);
  \\', 'path [arrow] (i) -- node [above,midway] {', yi, '} (y);
  \\', 'path [arrow] (g) -- (inter);
  \\', 'path [arrow] (i) -- (inter);
  \\', 'path [arrow] (inter) -- node [left,midway,xshift=1ex] {', inter, '} (y);
  \\', 'end{tikzpicture}
  \\', 'end{document} \n'))
  sink()
  cmd <- paste0('pdflatex ', tempfilepref, '.tex')
  system(cmd)
  cmd <- paste0('convert -density 300 ', tempfilepref, '.pdf -quality 90 ', 'intergraph.', filename, '.png')
  system(cmd)
  cat(paste0('![](', 'intergraph.', filename, '.png', ')'))
  cmd <- paste0('rm ../rmd/', tempfilepref, '.*')
  system(cmd)
  cat('\n')
}
```

# Summary

This page presents the results of analysis that have been done. Every section is one disease (phenotype of interest) and each subsection is the analysis for a particular intermediate trait.

Every subsection presents the following information.

* Comparison between raw $\beta$ and posterior mean of $\beta$ for computing PRS (This is not available now)
* Marginal association:
    - $Y \sim I$
    - $Y \sim G_i$
    - $Y \sim I + G_i$
* Interaction test results based on functional subsetting
* Follow up comment on the results (functional subsetting) if any
* Interaction test results based on significance subsetting (if run)
* Follow up comment on the results (significance subsetting) if any

# Coronary heart disease

## HDL

```{r, results='asis'}
config.hdl <- '../code/step2/config.wtccc-cad-hdl.yaml'
doAll(config.hdl)
listSubset(config.hdl, 'func_subset')
```

```{r, results='asis'}
listSubset(config.hdl, 'sig_subset')
```

## LDL

```{r, results='asis'}
config.ldl <- '../code/step2/config.wtccc-cad-ldl.yaml'
doAll(config.ldl)
listSubset(config.ldl, 'func_subset')
```

This [paper](http://circres.ahajournals.org/content/118/1/83) shows the strong genetic correlation between coronary heart disease and LDL and conditional FDR (on LDL; namely incorporating LDL p-value into FDR calculation) of CHD help to identify more signal out of GWAS follow-up study. Also, LDL has been shown to be causal factor of CHD, [paper](http://www.nejm.org.proxy.uchicago.edu/doi/full/10.1056/NEJMoa054013).

[*rs13465*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=13465) locates in the intronic region of ILF3. Its protective effect is masked by high LDL level.

```{r, results='asis'}
drawGraph('-', '-', '+', '+', 'cad-ldl') #
```

Interestingly, this [paper](https://www.spandidos-publications.com/mmr/4/3/511) reported that ILF3 (*rs2569512* was used to tag the gene) affects myocardial infarction risk only when LDL level is low (when LDL level is high, there is no association signal).

```{r, results='asis'}
listSubset(config.ldl, 'sig_subset')
```

Here, I list SNPs that have genome-wide significant p-value in GWAS and interaction p-value < 0.1

[*rs564398*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=564398) locates in the intronic region of CDKN2B-AS1 (CDKN2B antisense RNA 1). It is a known CHD locus [link](https://www.ncbi.nlm.nih.gov/gene/100048912). It protective effect goes away when LDL is high but there is no strong correlation between the locus and LDL level.

[*rs7865618*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=7865618) also locates in CDKN2B-AS1 (intronic region). It works in the same direction of *rs564398*.

[*rs7049105*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=7049105) also locates in CDKN2B-AS1 (intronic region). Its risk goes away when LDL level is high. This direction is also consistent with the previous two loci.

[*rs10965215*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=10965215) also locates in CDKN2B-AS1 (intronic region). The direction is the same as *rs7049105*.

[*rs10965219*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=7049105) also locates in CDKN2B-AS1 (intronic region). Same direction.

It seems that the effect of CDKN2B-AS1 on CHD is masked by high LDL in general. But no evidence support the interaction between CDKN2B-AS1 and LDL on CHD risk.

Here, I list the rest of SNPs that pass interaction p-value < 0.1

[*rs7250581*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=7250581) is in intergenic region but it has been confirmed in many studies across multiple populations. Also, the protective effect of the loci goes away when LDL is high.

[*rs2398486*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2398486) locates in the intronic region of [TMEM132D](http://www.uniprot.org/uniprot/Q14C87). It may not be the target gene. The direction is also the same.

[*rs2167512*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2167512) also locates in the intronic region of TMEM132D. Direction is the same.

[*rs2838756*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2838756) locates in intergenic region. The direction is still the same.

[*rs523096*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=523096) is another SNP in the intronic region of CDKN2B-AS1. Direction is the same.

```{r}
drawLD(c('rs564398', 'rs7865618', 'rs7049105', 'rs10965215', 'rs10965219', 'rs523096', 'rs2398486', 'rs2167512'), config.ldl)
```

In brief, all directions are (except one with fliped sign)

```{r, results='asis'}
drawGraph('-', 'no', '+', '+', 'cad-ldl2')  # gy, gi, yi
```

## Serum triglycerides concentration

```{r, results='asis'}
config.tg <- '../code/step2/config.wtccc-cad-tg.yaml'
doAll(config.tg)
listSubset(config.tg, 'func_subset')
```

This paper proposed that triglycerides level is also a causal factor of CHD, [paper](https://www-nature-com.proxy.uchicago.edu/articles/ng.2795).

[*rs4541125*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=4541125) shows most significant interaction (positive: i.e. as $I$ increases, the effect size of $G$ increases). In other word, the protective effect of $G$ goes away when $I$ is high. It locates in the intronic region of PITPNM3, which is reported to be under-presented in the proteomics of Acute Ischemic Strokes patients ([table 3 of the paper](https://pubs.acs.org/doi/full/10.1021/pr501324n?src=recsys)). Note that the triglycerides level is high in cases versus controls.

```{r, results='asis'}
drawGraph('-', '-', '+', '+', 'cad-tg')  # gy, gi, yi
```

```{r, results='asis'}
listSubset(config.tg, 'sig_subset')
```

No genome-wide significant SNP get interaction p-value smaller than 0.1. Here I list the ones with interaction p-value less than 0.1

[*rs2167512*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2167512) and [*rs2398486*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2398486) is the same as what we find in LDL. Same direction as before and here this SNP has a weak posterior effect on triglyceride level (negative). The direction is also the same. Namely, the protective effect (possibly through triglyceride level) is masked by high TG.

[*rs7250581*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2167512) is also found in LDL. Same direction.

```{r, results='asis'}
drawGraph('-', 'no(-)', '+', '+', 'cad-tg2')  # gy, gi, yi
```

## Leptin

```{r, results='asis'}
config.leptin <- '../code/step2/config.wtccc-cad-leptin.yaml'
doAll(config.leptin)
listSubset(config.leptin, 'func_subset')
```

# Crohn's disease

## C-reactive protein

It turns out that the GWAS signal goes away if limiting to loci which occur in WTCCC genotype data.

## Insulin-like growth factor

```{r, results='asis'}
config.igf1 <- '../code/step2/config.wtccc-crohn-igf1.yaml'
doAll(config.igf1)
listSubset(config.igf1, 'func_subset')
```

## White blood cell count

```{r, results='asis'}
config.wbc <- '../code/step2/config.wtccc-crohn-wbc.yaml'
doAll(config.wbc)
listSubset(config.wbc, 'func_subset')
```

[*rs7775228*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=7775228) is a tag SNP for HLA DQ2.2 haplotype [link](https://www.snpedia.com/index.php/Rs7775228).

Interestingly, when adding a sparse prior on the effect size, two SNPs ([*rs4379550*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=4379550) and [*rs11145756*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=rs11145756) are both located in the intronic region of SEC16A. And their interaction terms is negative, indicating that the SNP and intermediate trait act in redundant manner.

```{r, results='asis'}
drawGraph('+', '+', '+', '-', 'crohn-wbc')  # gy, gi, yi
```

```{r, results='asis'}
listSubset(config.wbc, 'sig_subset')
```

There are four SNPs that hit genome-wide significance with interaction p-value < 0.102. I list them first.

[*rs2076756*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2076756) locates in the intronic region of [NOD2](https://en.wikipedia.org/wiki/NOD2) which is an IBD gene. Its risk effect goes away when the WBC is high. It has slightly positive effect on WBC, which indicate that its effect on CD does not through WBC.

[*rs16869934*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=16869934) locates in intergenic region. The corresponding gene is unclear. The protective effect goes away when the WBC is high.

[*rs10512734*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=10512734) is in intergenic region. The corresponding gene is unknown. Its protective effect goes away when WBC is high.

[*rs2066843*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2066843) locates in NOD2 (synonymous). It acts the same way as *rs2076756*.

The following list the rest of SNPs with interaction p-value < 0.1

[*rs3816769*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=3816769) locates in the intronic region of [STAT3](https://www.uniprot.org/uniprot/P40763) is *signal transducer and transcription activator that mediates cellular responses to interleukins, KITLG/SCF, LEP and other growth factors*. Interestingly, its protective effect becomes stronger when WBC is high.

[*rs1026916*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1026916), [*rs7211777*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=7211777) also locate in intronic region of STAT3. And they act the same direction as *rs3816769*.

[*rs8111071*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=8111071) locates in the intronic region of [RSPH6A](https://www.ncbi.nlm.nih.gov/gene/81492). Its risk is enhanced when WBC is high.

[*rs9469615*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=9469615) locates in intergenic region. The gene is unclear. The interaction enhance the effect.

This SNP high slightly modest interaction (p-value = 0.1111). [*rs4807569*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=4807569) locates in intronic region of SBNO2. Its effect is also got enhanced. Note that it also has a strong positive effect on  WBC.

In summary, both masking and enhancing interactions exist.

```{r}
drawLD(c('rs2076756', 'rs2066843', 'rs3816769', 'rs1026916', 'rs7211777'), config.wbc)
```

# Type II diabetes

## Fast glucose

```{r, results='asis'}
config.fastglu <- '../code/step2/config.wtccc-t2d-fastglu.yaml'
doAll(config.fastglu)
listSubset(config.fastglu, 'func_subset')
```

[*rs10509645*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=10509645) locates in the intronic region of [IDE](http://www.uniprot.org/uniprot/P14735).

[*rs5015480*](https://www.snpedia.com/index.php/Rs5015480) (gene: [HHEX](http://www.uniprot.org/uniprot/Q03014)).

For [*rs1243150*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=275775) and [*rs1243149*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=275774), the gene is unknown.

[*rs7923866*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=rs7923866) is also in the vicinity of the HHEX (homeobox, hematopoietically expressed) and IDE (insulin-degrading enzyme) [link](https://www.nature.com/articles/nature05911).

[*rs760233*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=760233) locates in the intronic region of [TTLL5](https://www.uniprot.org/uniprot/Q6EMB2), which is related to retinal development and has not been reported to be associated with T2D.

<!--

[*10811661*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=10811661) is strongly associated with T2D [link](https://www.snpedia.com/index.php/Rs10811661) which is located at upstream of CDKN2A/B. [This paper](http://diabetes.diabetesjournals.org/content/early/2018/02/05/db17-1055) proposed that *CDKN2A/B locus SNPs may impact T2D risk by modulating islet gene expression and beta
cell proliferation*.

[*rs2383208*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2383208)  also locates near CDKN2A/2B [link](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0026953). -->

Overall, all the SNPs mentioned above have the same direction. Namely the protective effect is masked if the fast glucose background level is high.

```{r, results='asis'}
drawGraph('-', '-', '+', '+', 't2d-fastglu')  # gy, gi, yi
```

```{r, results='asis'}
listSubset(config.fastglu, 'sig_subset')
```

*rs5015480* and *rs7923866* have been shown in `func_subset` analysis and they locate in intergenic region. They are near HHEX and IDE. This region has been confirmed by multiple studies.

[*rs11075123*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=11075123) locates in intergenic region. The directions are
```{r, results='asis'}
drawGraph('-', '-', '+', '-', 't2d-fastglu2')  # gy, gi, yi
```

Interestingly, the direction is not the same as before. Namely the protective effect is stronger if fast glucose level is high.

[*rs2930291*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2930291) locates in the intronic region of [CCDC33](http://www.uniprot.org/uniprot/Q8N5R6). Not sure about its function. The protective effect is stronger when fast glucose level is high, which is the same as *rs11075123* but the opposite of the others.

```{r}
drawLD(c('rs5015480', 'rs7923866'), config.fastglu)
```

## Fast insulin

```{r, results='asis'}
config.fastinsulin <- '../code/step2/config.wtccc-t2d-fastinsulin.yaml'
doAll(config.fastinsulin)
listSubset(config.fastinsulin, 'func_subset')
```

[*rs2155105*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2155105) locates in the intronic region of [RAB38](http://www.uniprot.org/uniprot/P57729). It is associated with albuminuria in T2D.

rs1996913, rs17606320, rs7544694, rs7543693, rs12173920, rs16895362,rs2049316, rs6864348 have almost no signal in $I \sim G$.

[*rs3734262*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=3734262) locates in the intronic region of [UHRF1](http://www.uniprot.org/uniprot/Q96T88) binding protein 1 ([UHRF1BP1](http://www.uniprot.org/uniprot/Q6BDS2))

[*rs9368834*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=9368834) locates in the intronic region of [ANKS1A](https://www.uniprot.org/uniprot/Q92625). It has been reported that this gene is associated with T2D in obese individuals.

[*rs2139980*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2139980). It direction is not the same as others.

All SNPs but rs2139980 mentioned has protective effect on T2D and when background fast insulin is high, the protective effects become smaller. For rs2139980, its protective effect becomes even stronger.

```{r, results='asis'}
drawGraph('-', '-', '+', '+', 't2d-fastinsulin')  # gy, gi, yi
```

```{r, results='asis'}
listSubset(config.fastinsulin, 'sig_subset')
```

No genomve-wide significant hits.

[*rs8012854*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=8012854) locates in LOC105370657 upstream 2kb. Unsure function. Its protective effect goes away when fast insulin is high.

[*rs440646*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=440646) locates in the intronic region of [HRH1](https://www.uniprot.org/uniprot/P35367). *Chemical genetic identification of the histamine H1 receptor as a stimulator of insulin-induced adipogenesis.* [link](https://www.wikigenes.org/e/ref/e/15271349.html). Its risk goes up when fast insulin is high.

*rs11075123* also shows up in `sig_subset` analysis. Here its protective effect becomes stronger when fast insulin is high, which is the same as previous result.

[*rs7961581*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=7961581) is in intergenic region. Nearest genes are [TSPAN8](http://www.uniprot.org/uniprot/P19075), [LGR5](http://www.uniprot.org/uniprot/O75473). The risk is higher when the fast insulin is high.

Overall, the effect is always boosted when fast insulin is high but *rs8012854* does not follow it (its effect is masked by high fast insulin).

```{r, results='asis'}
drawGraph('-/+', 'no', '+', '-/+', 't2d-fastinsulin2')  # gy, gi, yi
```
