---
title: "Results"
# author: Yanyu Liang
date: "`r format(Sys.time(), '%d %B, %Y')`"
---

```{r, echo=FALSE}
library(yaml)
reportThre <- function(meta) {
  threshold <- meta$`functional-subsetting`$`pval-theshold`
  cat('The threhold used for extracting functionally consistent variant is', threshold, '\n')
}
plotImage <- function(meta) {
  name <- paste0('../code/step2/output-direction_XT_YI/', meta$`TRUE`, '__', meta$x1, '__', meta$x2, '/direction_yi.png')
  cat(paste0('![](', name, ')'), '\n')
  cat('\n')
}
listReports <- function(meta) {
  cat(paste0('1. Comparison of Beta [here](', 'https://rawgit.com/liangyy/phenotype-epistasis/master/code/step1/output-run_step1/', meta$`TRUE`, '__', meta$x1, '__', meta$x2, '/prs_report.html', ')'), '\n')
  cat(paste0('2. Marginal directions [here](https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-direction_XT_YI/', meta$`TRUE`, '__', meta$x1, '__', meta$x2, '/direction_report.html)'), '\n')
  cat(paste0('3. Interaction test results are [here](https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-run_step2/', meta$`TRUE`, '__', meta$x1, '__', meta$x2, '/inter_report.func_subset.html)'), '\n')
}
doAll <- function(filename) {
  meta <- yaml.load_file(filename)
  reportThre(meta)
  plotImage(meta)
  listReports(meta)
}
```

# Summary

This page presents the results of analysis that have been done. Every section is one disease (phenotype of interest) and each subsection is the analysis for a particular intermediate trait.

Every subsection presents the following information.

* Comparison between raw $\beta$ and posterior mean of $\beta$ for computing PRS
* Marginal association:
    - $Y \sim I$
    - $Y \sim G_i$
    - $Y \sim I + G_i$
* Interaction test results based on functional subsetting
* Follow up comment on the results if any

# Coronary heart disease

## HDL

```{r, results='asis'}
config.hdl <- '../code/step2/config.wtccc-cad-hdl.yaml'
doAll(config.hdl)
```

## LDL

```{r, results='asis'}
config.ldl <- '../code/step2/config.wtccc-cad-ldl.yaml'
doAll(config.ldl)
```

This [paper](http://circres.ahajournals.org/content/118/1/83) shows the strong genetic correlation between coronary heart disease and LDL and conditional FDR (on LDL; namely incorporating LDL p-value into FDR calculation) of CHD help to identify more signal out of GWAS follow-up study. Also, LDL has been shown to be causal factor of CHD, [paper](http://www.nejm.org.proxy.uchicago.edu/doi/full/10.1056/NEJMoa054013).

[*rs13465*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=13465) locates in the intronic region of ILF3. Its protective effect is masked by high LDL level.

```{r, echo=FALSE, engine="tikz", fig.ext='pdf', fig.align='center', out.width=150, out.height=300}
\usetikzlibrary{fit,calc,positioning,decorations.pathreplacing,matrix}
\usetikzlibrary{positioning}
\usetikzlibrary{shapes, arrows, calc, positioning}
\begin{tikzpicture}[auto]
  \tikzstyle{node}=[draw, rectangle, text centered, node distance=5em]
  \tikzstyle{arrow}=[draw, ->]
  \node [node] (y) {$Y$};
  \node [node, below left of=y] (g) {$G$};
  \node [node, below right of=y] (i) {$I$};
  \path [arrow] (g) -- node [above,midway] {-} (y);
  \path [arrow] (g) -- node [above,midway] {-} (i);
  \path [arrow] (i) -- node [above,midway] {+} (y);
\end{tikzpicture}
```

Interestingly, this [paper](https://www.spandidos-publications.com/mmr/4/3/511) reported that ILF3 (*rs2569512* was used to tag the gene) affects myocardial infarction risk only when LDL level is low (when LDL level is high, there is no association signal).

## Serum triglycerides concentration

```{r, results='asis'}
config.tg <- '../code/step2/config.wtccc-cad-tg.yaml'
doAll(config.tg)
```

This paper proposed that triglycerides level is also a causal factor of CHD, [paper](https://www-nature-com.proxy.uchicago.edu/articles/ng.2795).

[*rs4541125*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=4541125) shows most significant interaction (positive: i.e. as $I$ increases, the effect size of $G$ increases). In other word, the protective effect of $G$ goes away when $I$ is high. It locates in the intronic region of PITPNM3, which is reported to be under-presented in the proteomics of Acute Ischemic Strokes patients ([table 3 of the paper](https://pubs.acs.org/doi/full/10.1021/pr501324n?src=recsys)). Note that the triglycerides level is high in cases versus controls.

```{r, echo=FALSE, engine="tikz", fig.ext='pdf', fig.align='center', out.width=150, out.height=300}
\usetikzlibrary{fit,calc,positioning,decorations.pathreplacing,matrix}
\usetikzlibrary{positioning}
\usetikzlibrary{shapes, arrows, calc, positioning}
\begin{tikzpicture}[auto]
  \tikzstyle{node}=[draw, rectangle, text centered, node distance=5em]
  \tikzstyle{arrow}=[draw, ->]
  \node [node] (y) {$Y$};
  \node [node, below left of=y] (g) {$G$};
  \node [node, below right of=y] (i) {$I$};
  \path [arrow] (g) -- node [above,midway] {-} (y);
  \path [arrow] (g) -- node [above,midway] {-} (i);
  \path [arrow] (i) -- node [above,midway] {+} (y);
\end{tikzpicture}
```

# Crohn's disease

## C-reactive protein

It turns out that the GWAS signal goes away if limiting to loci which occur in WTCCC genotype data.

## Insulin-like growth factor

```{r, results='asis'}
config.igf1 <- '../code/step2/config.wtccc-crohn-igf1.yaml'
doAll(config.igf1)
```

## White blood cell count

```{r, results='asis'}
config.wbc <- '../code/step2/config.wtccc-crohn-wbc.yaml'
doAll(config.wbc)
```

Interestingly, when adding a sparse prior on the effect size, two SNPs ([*rs4379550*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=4379550) and [*rs11145756*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=rs11145756) are both located in the intronic region of SEC16A. And their interaction terms is negative, indicating that the SNP and intermediate trait act in redundant manner.

```{r, echo=FALSE, engine="tikz", fig.ext='pdf', fig.align='center', out.width=150, out.height=300}
\usetikzlibrary{fit,calc,positioning,decorations.pathreplacing,matrix}
\usetikzlibrary{positioning}
\usetikzlibrary{shapes, arrows, calc, positioning}
\begin{tikzpicture}[auto]
  \tikzstyle{node}=[draw, rectangle, text centered, node distance=5em]
  \tikzstyle{arrow}=[draw, ->]
  \node [node] (y) {$Y$};
  \node [node, below left of=y] (g) {$G$};
  \node [node, below right of=y] (i) {$I$};
  \path [arrow] (g) -- node [above,midway] {+} (y);
  \path [arrow] (g) -- node [above,midway] {+} (i);
  \path [arrow] (i) -- node [above,midway] {+} (y);
\end{tikzpicture}
```
