<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Computation</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-4.5.0/css/font-awesome.min.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Phenotypic Genetic Background Effect</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="data.html">Data</a>
</li>
<li>
  <a href="computation.html">Computation</a>
</li>
<li>
  <a href="implementation.html">Implementation</a>
</li>
<li>
  <a href="results.html">Results</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/liangyy/phenotype-epistasis">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Computation</h1>
<h4 class="date"><em>16 February, 2018</em></h4>

</div>


<p><span class="math display">\[
\newcommand\logit{\text{logit}}  
\]</span></p>
<div id="computing-intermediate-phenotype" class="section level1">
<h1><span class="header-section-number">1</span> Computing intermediate phenotype</h1>
<p>The predicted complex trait risk by GWAS summary statistic is used as intermediate genetic trait. LD pruning + p-value thresholding (P+T) is widely used but suffer from lack of effective SNPs.</p>
<div id="pt" class="section level2">
<h2><span class="header-section-number">1.1</span> P+T</h2>
<p>LD pruning (<span class="math inline">\(r^2\)</span> &lt; 0.2) followed by p-value thresholding with various threholds optimized by validation prediction accurarcy (<span class="math inline">\(R^2\)</span>). One implementation can be found at <a href="http://prsice.info/">PRSice</a></p>
</div>
<div id="ldpred" class="section level2">
<h2><span class="header-section-number">1.2</span> LDpred</h2>
<p>Consider LD effect in each block (<span class="math inline">\(\sim M / 3000\)</span>) with various prior fraction of causal variants optimized by validation accuracy. One implementation can be found at <a href="https://github.com/bvilhjal/ldpred">here</a> (also see <a href="http://linkinghub.elsevier.com/retrieve/pii/S0002-9297(15)00365-1">paper</a>)</p>
<p>Note that LDpred also includes codes for P+T strategy.</p>
</div>
<div id="validation" class="section level2">
<h2><span class="header-section-number">1.3</span> Validation</h2>
<p>The difficulty is that we don’t have validation data sets to determine the hyper-parameters needed for the calculation of polygenic risk score. One strategy is to see if the order of PRS is stable across a range of hyper-parameters. Note that is can be benchmarked using disease phenotypes of which we have individual genotypes, <em>i.e.</em> WTCCC 7 diseases.</p>
</div>
</div>
<div id="testing-interaction" class="section level1">
<h1><span class="header-section-number">2</span> Testing interaction</h1>
<div id="model" class="section level2">
<h2><span class="header-section-number">2.1</span> Rationale</h2>
The epistasis between genetic background and the given locus is defined as the situation where the contribution of genotype (<span class="math inline">\(G_i\)</span>) to liability depends not only on the genotype itself but also on the genetic background which is measured by the genetic component of the selected intermediate phenotype (<span class="math inline">\(I\)</span>). In particular, the following model is used
<span class="math display">\[\begin{align*}
  \logit(\Pr(Y = 1|G_i, I, \alpha, \beta, \gamma)) &amp;= \alpha + \mathbf{p}(I)^T \cdot G_i + \gamma I 
\end{align*}\]</span>
<p>, where <span class="math inline">\(\mathbf{p}: I \rightarrow \beta\)</span> which takes <span class="math inline">\(I\)</span> and maps to the effect size in terms of the contribution to log odds ratio. Note that the representation of <span class="math inline">\(G_i\)</span> can be vector or loading depends on how the genetics is specified (see <a href="#scenarios">here</a>). <span class="math inline">\(\gamma\)</span> is the contribution of <span class="math inline">\(I\)</span> to liability which should be universal across all genotypes and all loci. <span class="math inline">\(\alpha\)</span> is the baseline liability of the trait which should also be share across loci.</p>
</div>
<div id="scenarios" class="section level2">
<h2><span class="header-section-number">2.2</span> Scenarios</h2>
<p>The following four scenarios are considered (suppose a is risk allele):</p>
<ol style="list-style-type: decimal">
<li>additive effect:
<span class="math display">\[\begin{align*}
G_i &amp;= \begin{cases}
  0 &amp; \text{, if genotype is AA} \\
  1 &amp; \text{, if genotype is Aa} \\
  2 &amp; \text{, if genotype is aa} 
\end{cases}
  \end{align*}\]</span>
, and the contribution of liability is <span class="math inline">\(p(I) G_i\)</span>, specifically <span class="math inline">\(p(I)\)</span> is the baseline contribution function (the non-risk allele contribution).</li>
<li>dominant(recessive) effect:
<span class="math display">\[\begin{align*}
G_i &amp;= \begin{cases}
  1 &amp; \text{, if genotype is aa or Aa (aa)} \\
  0 &amp; \text{, if genotype is AA (AA or Aa)}
\end{cases}
  \end{align*}\]</span>
, and <span class="math inline">\(p_i(I)\)</span> is the corresponding contribution function of risk allele case.</li>
<li>free effect:
<span class="math display">\[\begin{align*}
G_i &amp;= \begin{cases}
  (0, 0)^T &amp; \text{, if genotype is AA} \\
  (0, 1)^T &amp; \text{, if genotype is Aa} \\
  (1, 0)^T &amp; \text{, if genotype is aa}
\end{cases}
  \end{align*}\]</span>
, namely every genotype has a specific contribution function. Note that the effect size is respective to the reference allele (non-risk allele). The effect of non-risk allele is obserbed into <span class="math inline">\(\alpha + \gamma I\)</span>.</li>
</ol>
<!-- ## Procedure -->
<!-- 1. determine the risk allele by vanilla GWAS -->
<!-- 2. estimate $\alpha$ and $\gamma$. Namely perform regression $Y \sim 1 + I$ with link function logistic and determine the positive direction of $I$ (the direction that decreases liability) -->
<!-- 3. estimate the parameters in $\mathbf{p}(I)$. Specifically, let $p_i$ as degree $n$ polynomial. In practice $n = 1, 2$ -->
<!-- 4. test if the parameters of $p_i$ other than zero order are significantly different from zero -->
<!-- 5. identify loci with protective epistasis, namely the effect of risk allele goes down if the genetic background goes to the positive direction -->
</div>
<div id="interpretation-under-linear-model" class="section level2">
<h2><span class="header-section-number">2.3</span> Interpretation under linear model</h2>
We focus on the following model
<span class="math display">\[\begin{align*}
  \logit(\Pr(Y = 1 | G_i, I)) &amp;= \alpha + \beta_1 G_i + \beta_2 I + \gamma I G_i 
\end{align*}\]</span>
<p>Then, <span class="math inline">\(p(I) = \beta_1 + \gamma I\)</span> in this case.</p>
</div>
<div id="functionally-consistent-snp-set" class="section level2">
<h2><span class="header-section-number">2.4</span> Functionally consistent SNP set</h2>
<p>Let’s consider the set of SNPs such that the association signals can be explained by <span class="math inline">\(Y \sim I \sim G_i\)</span>. Namely, restrict to the following four scenaros.</p>
<table style="width:53%;">
<colgroup>
<col width="16%" />
<col width="11%" />
<col width="12%" />
<col width="12%" />
</colgroup>
<thead>
<tr class="header">
<th align="center"> </th>
<th align="center">Y ~ I</th>
<th align="center">Y ~ Gi</th>
<th align="center">I ~ Gi</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center"><strong>case1</strong></td>
<td align="center">+</td>
<td align="center">+</td>
<td align="center">+</td>
</tr>
<tr class="even">
<td align="center"><strong>case2</strong></td>
<td align="center">+</td>
<td align="center">-</td>
<td align="center">-</td>
</tr>
<tr class="odd">
<td align="center"><strong>case3</strong></td>
<td align="center">-</td>
<td align="center">+</td>
<td align="center">-</td>
</tr>
<tr class="even">
<td align="center"><strong>case4</strong></td>
<td align="center">-</td>
<td align="center">-</td>
<td align="center">+</td>
</tr>
</tbody>
</table>
<p>The hypothesis is that if <span class="math inline">\(G_i\)</span> acts on <span class="math inline">\(Y\)</span> via <span class="math inline">\(I\)</span>, there might be epistatic effect between <span class="math inline">\(I\)</span> and <span class="math inline">\(G_i\)</span>. Namely, the effect of <span class="math inline">\(G_i\)</span> depends on the background level of <span class="math inline">\(I\)</span>.</p>
</div>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
