<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Data</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-1.1/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-1.1/highlight.js"></script>
<link href="site_libs/font-awesome-4.5.0/css/font-awesome.min.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Phenotypic Genetic Background Effect</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="data.html">Data</a>
</li>
<li>
  <a href="computation.html">Computation</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/liangyy/phenotype-epistasis">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Data</h1>
<h4 class="date"><em>21 January, 2018</em></h4>

</div>


<div id="intermediate-gwas" class="section level1">
<h1><span class="header-section-number">1</span> Intermediate GWAS</h1>
<div id="blood-cell-phenotype" class="section level2">
<h2><span class="header-section-number">1.1</span> Blood cell phenotype</h2>
<ul>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/AstleWJ_27863252_GCST004602">Data</a></li>
<li><a href="http://www.cell.com/cell/fulltext/S0092-8674(16)31463-5">Paper</a></li>
<li>Meta-data</li>
</ul>
<table>
<colgroup>
<col width="4%" />
<col width="19%" />
<col width="15%" />
<col width="13%" />
<col width="17%" />
<col width="19%" />
<col width="9%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">id</th>
<th align="center">phenotype</th>
<th align="center">num.of.traits</th>
<th align="center">sample.size</th>
<th align="center">num.of.variants</th>
<th align="center">heritability.mean</th>
<th align="center">ancestry</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">1</td>
<td align="center">blood cell traits</td>
<td align="center">36</td>
<td align="center">173480</td>
<td align="center">29500000</td>
<td align="center">10%-30%</td>
<td align="center">european</td>
</tr>
</tbody>
</table>
<p>The phenotype information is as follow (Table S1)</p>
<pre><code>## 
## Attaching package: &#39;dplyr&#39;</code></pre>
<pre><code>## The following objects are masked from &#39;package:stats&#39;:
## 
##     filter, lag</code></pre>
<pre><code>## The following objects are masked from &#39;package:base&#39;:
## 
##     intersect, setdiff, setequal, union</code></pre>
<div style="border: 1px solid #ddd; padding: 5px; overflow-y: scroll; height:500px; overflow-x: scroll; width:1000px; ">
<table class="table" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
Cell.Type
</th>
<th style="text-align:left;">
Standard.Abbreviation
</th>
<th style="text-align:left;">
Long.Name
</th>
<th style="text-align:left;">
Unit
</th>
<th style="text-align:left;">
Description
</th>
<th style="text-align:left;">
Measured…Derived
</th>
<th style="text-align:left;">
Determination
</th>
<th style="text-align:left;">
Measured…Derived.1
</th>
<th style="text-align:left;">
Determination.1
</th>
<th style="text-align:left;">
Previous.GWAS
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Platelet
</td>
<td style="text-align:left;">
PLT#
</td>
<td style="text-align:left;">
Platelet count
</td>
<td style="text-align:left;">
per nL
</td>
<td style="text-align:left;">
Count of platelets per unit volume of blood
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Impedance
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Flow cytometry gate (impedance for missing data points)
</td>
<td style="text-align:left;">
Yes
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
MPV
</td>
<td style="text-align:left;">
Mean platelet volume
</td>
<td style="text-align:left;">
fL
</td>
<td style="text-align:left;">
Mean volume of platelets
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(PCT/PLT#)×10000
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(PCT/PLT#)×10000
</td>
<td style="text-align:left;">
Yes
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
PDW
</td>
<td style="text-align:left;">
Platelet distribution width
</td>
<td style="text-align:left;">
fL
</td>
<td style="text-align:left;">
The spread of the platelet volume distribution. Note that Sysmex and Coulter use different statistics to measure spread.
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Impedance: Coefficient of variation of platelet volume distribution
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Impedance: width at 20% peak height of platelet volume histogram.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
PCT
</td>
<td style="text-align:left;">
Plateletcrit
</td>
<td style="text-align:left;">
%
</td>
<td style="text-align:left;">
Volume fraction of blood occupied by platelets
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Impedance
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Impedance
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Mature red cell
</td>
<td style="text-align:left;">
RBC#
</td>
<td style="text-align:left;">
Red blood cell count
</td>
<td style="text-align:left;">
per pL
</td>
<td style="text-align:left;">
Count of red blood cells per unit volume of blood
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Impedance
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Impedance
</td>
<td style="text-align:left;">
Yes
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
MCV
</td>
<td style="text-align:left;">
Mean corpuscular volume
</td>
<td style="text-align:left;">
fL
</td>
<td style="text-align:left;">
Mean volume of red blood cells
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(HCT/RBC#)×10
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(HCT/RBC#)×10
</td>
<td style="text-align:left;">
Yes
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
HCT
</td>
<td style="text-align:left;">
Hematocrit
</td>
<td style="text-align:left;">
%
</td>
<td style="text-align:left;">
Volume fraction of blood occupied by red cells
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Impedance
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Impedance
</td>
<td style="text-align:left;">
Yes
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
MCH
</td>
<td style="text-align:left;">
Mean corpuscular hemoglobin
</td>
<td style="text-align:left;">
pg
</td>
<td style="text-align:left;">
Average mass of hemoglobin per red cell
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(HGB/RBC#)×10
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(HGB/RBC#)×10
</td>
<td style="text-align:left;">
Yes
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
MCHC
</td>
<td style="text-align:left;">
Mean corpuscular hemoglobin concentration
</td>
<td style="text-align:left;">
g/dL
</td>
<td style="text-align:left;">
Concentration of hemoglobin with respect to unit of volume occupied by red cells
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(HGB/HCT)×100
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(HGB/HCT)×100
</td>
<td style="text-align:left;">
Yes
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
HGB
</td>
<td style="text-align:left;">
Hemoglobin concentration
</td>
<td style="text-align:left;">
g/dL
</td>
<td style="text-align:left;">
Concentration of hemoglobin with respect to unit of volume of blood
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Light absorbance
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Light absorbance
</td>
<td style="text-align:left;">
Yes
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
RDW
</td>
<td style="text-align:left;">
Red cell distribution width
</td>
<td style="text-align:left;">
fL
</td>
<td style="text-align:left;">
Coefficient of variation of red cell volume distribution
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
CV of impedance measured red cell volume distribution
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
CV of impedance measured red cell volume distribution
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Immature red cell
</td>
<td style="text-align:left;">
RET#
</td>
<td style="text-align:left;">
Reticulocyte count
</td>
<td style="text-align:left;">
pL
</td>
<td style="text-align:left;">
Count of reticulocytes per unit volume of blood
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(RET%×RBC#)/100
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(RET%×RBC#)/100
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
RET%
</td>
<td style="text-align:left;">
Reticulocyte fraction of red cells
</td>
<td style="text-align:left;">
%
</td>
<td style="text-align:left;">
Percentage of red blood cells that are reticulocytes
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Flow cytometry/impedance gates
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Flow cytometry gates
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
IRF
</td>
<td style="text-align:left;">
Immature fraction of reticulocytes
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
Fraction of reticulocytes with high RNA content, as measured by light scatter
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
HLSR#/RET#
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Flow cytometry gates
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
HLSR#
</td>
<td style="text-align:left;">
High light scatter reticulocyte count
</td>
<td style="text-align:left;">
per pL
</td>
<td style="text-align:left;">
Count of high RNA content (immature) reticulocytes per unit volume of blood
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(HLSR%×RBC#)/100%
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
IRF×RET#
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
HLSR%
</td>
<td style="text-align:left;">
High light scatter reticulocyte percentage of red cells
</td>
<td style="text-align:left;">
%
</td>
<td style="text-align:left;">
Immature reticulocyte count as a percentage of red blood cell count
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Flow cytometry/impedance gates
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(HLSR#/RBC#)×100%
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Myeloid white cell
</td>
<td style="text-align:left;">
MONO#
</td>
<td style="text-align:left;">
Monocyte count
</td>
<td style="text-align:left;">
per nL
</td>
<td style="text-align:left;">
Count of monocytes per unit volume of blood
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(MONO%×WBC#)/100%
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(MONO%×WBC#)/100%
</td>
<td style="text-align:left;">
Yes
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
NEUT#
</td>
<td style="text-align:left;">
Neutrophil count
</td>
<td style="text-align:left;">
per nL
</td>
<td style="text-align:left;">
Count of neutrophils per unit volume of blood
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(NEUT%×WBC#)/100%
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(NEUT%×WBC#)/100%
</td>
<td style="text-align:left;">
Yes
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
EO#
</td>
<td style="text-align:left;">
Eosinophil count
</td>
<td style="text-align:left;">
per nL
</td>
<td style="text-align:left;">
Count of eosinophils per unit volume of blood
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(EO%×WBC#)/100%
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(EO%×WBC#)/100%
</td>
<td style="text-align:left;">
Yes
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
BASO#
</td>
<td style="text-align:left;">
Basophil count
</td>
<td style="text-align:left;">
per nL
</td>
<td style="text-align:left;">
Count of basophils per unit volume of blood
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(BASO%×WBC#)/100%
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(BASO%×WBC#)/100%
</td>
<td style="text-align:left;">
Yes
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
(NEUT+EO)#
</td>
<td style="text-align:left;">
Sum neutrophil eosinophil counts
</td>
<td style="text-align:left;">
per nL
</td>
<td style="text-align:left;">
Aggregate count of neutrophils and eosinophils per unit volume of blood.
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
NEUT#+EO#
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
NEUT#+EO#
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
(EO+BASO)#
</td>
<td style="text-align:left;">
Sum eosinophil basophil counts
</td>
<td style="text-align:left;">
per nL
</td>
<td style="text-align:left;">
Aggregate count of eosinophils and basophils per unit volume of blood
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
EO#+BASO#
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
EO#+BASO#
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
(BASO+NEUT)#
</td>
<td style="text-align:left;">
Sum basophil neutrophil counts
</td>
<td style="text-align:left;">
per nL
</td>
<td style="text-align:left;">
Aggregate count of basophils and neutrophils per unit volume of blood
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
BASO#+NEUT#
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
BASO#+NEUT#
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
GRAN#
</td>
<td style="text-align:left;">
Granulocyte count
</td>
<td style="text-align:left;">
per nL
</td>
<td style="text-align:left;">
Aggregate count of granulocytes per unit volume of blood
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
NEUT#+BASO#+EO#
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
NEUT#+BASO#+EO#
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
NEUT%GRAN
</td>
<td style="text-align:left;">
Neutrophil percentage of granulocytes
</td>
<td style="text-align:left;">
%
</td>
<td style="text-align:left;">
Percentage of granulocytes that are neutrophils
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(NEUT#/GRAN#)×100%
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(NEUT#/GRAN#)×100%
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
BASO%GRAN
</td>
<td style="text-align:left;">
Basophil percentage of granulocytes
</td>
<td style="text-align:left;">
%
</td>
<td style="text-align:left;">
Percentage of granulocytes that are basophils
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(BASO#/GRAN#)×100%
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(BASO#/GRAN#)×100%
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
EO%GRAN
</td>
<td style="text-align:left;">
Eosinophil percentage of granulocytes
</td>
<td style="text-align:left;">
%
</td>
<td style="text-align:left;">
Percentage of granulocytes that are eosinophils
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(EO#/GRAN#)×100%
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(EO/#GRAN#)×100%
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
MYELOID#
</td>
<td style="text-align:left;">
Myeloid white cell count
</td>
<td style="text-align:left;">
per nL
</td>
<td style="text-align:left;">
Aggregate count of myeloid white cells
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
NEUT#+BASO#+EO#+MONO#
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
NEUT#+BASO#+EO#+MONO#
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
GRAN%MYELOID
</td>
<td style="text-align:left;">
Granulocyte percentage of myeloid white cells
</td>
<td style="text-align:left;">
%
</td>
<td style="text-align:left;">
Percentage of myeloid white cells that are granulocytes
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(GRAN#/MYELOID#)×100%
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(GRAN#/MYELOID#)×100%
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Lymphoid white cell
</td>
<td style="text-align:left;">
LYMPH#
</td>
<td style="text-align:left;">
Lymphocyte count
</td>
<td style="text-align:left;">
per nL
</td>
<td style="text-align:left;">
Aggregate count of lymphoid cells per unit volume of blood
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(LYMPH%×WBC#)/100%
</td>
<td style="text-align:left;">
Derived
</td>
<td style="text-align:left;">
(LYMPH%×WBC#)/100%
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Compound white cell
</td>
<td style="text-align:left;">
WBC#
</td>
<td style="text-align:left;">
White blood cell count
</td>
<td style="text-align:left;">
per nL
</td>
<td style="text-align:left;">
Aggregate count of white cells per unit volume of blood
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Impedance
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Flow cytometry gates
</td>
<td style="text-align:left;">
Yes
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
MONO%
</td>
<td style="text-align:left;">
Monocyte percentage of white cells
</td>
<td style="text-align:left;">
%
</td>
<td style="text-align:left;">
Percentage of white cells that are monocytes
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Flow cytometry gates
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Flow cytometry gates
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
NEUT%
</td>
<td style="text-align:left;">
Neutrophil percentage of white cells
</td>
<td style="text-align:left;">
%
</td>
<td style="text-align:left;">
Percentage of white cells that are neutrophils
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Flow cytometry gates
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Flow cytometry gates
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
EO%
</td>
<td style="text-align:left;">
Eosinophil percentage of white cells
</td>
<td style="text-align:left;">
%
</td>
<td style="text-align:left;">
Percentage of white cells that are eosinophils
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Flow cytometry gates
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Flow cytometry gates
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
BASO%
</td>
<td style="text-align:left;">
Basophil percentage of white cells
</td>
<td style="text-align:left;">
%
</td>
<td style="text-align:left;">
Percentage of white cells that are basophils
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Flow cytometry gates
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Flow cytometry gates
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
LYMPH%
</td>
<td style="text-align:left;">
Lymphocyte percentage of white cells
</td>
<td style="text-align:left;">
%
</td>
<td style="text-align:left;">
Percentage of white cells that are lymphocytes
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Flow cytometry gates
</td>
<td style="text-align:left;">
Measured
</td>
<td style="text-align:left;">
Flow cytometry gates
</td>
<td style="text-align:left;">
</td>
</tr>
</tbody>
</table>
</div></li>
</ul>
</div>
<div id="circulating-metabolites" class="section level2">
<h2><span class="header-section-number">1.2</span> Circulating metabolites</h2>
<ul>
<li><a href="http://www.computationalmedicine.fi/data#NMR_GWAS">Data</a></li>
<li><a href="https://www.nature.com/articles/ncomms11122">Paper</a></li>
<li>Meta-data</li>
</ul>
<table>
<colgroup>
<col width="4%" />
<col width="23%" />
<col width="14%" />
<col width="12%" />
<col width="16%" />
<col width="18%" />
<col width="9%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">id</th>
<th align="center">phenotype</th>
<th align="center">num.of.traits</th>
<th align="center">sample.size</th>
<th align="center">num.of.variants</th>
<th align="center">heritability.mean</th>
<th align="center">ancestry</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">2</td>
<td align="center">circulating metabolites</td>
<td align="center">123</td>
<td align="center">24925</td>
<td align="center">3.9e+07</td>
<td align="center">NA</td>
<td align="center">european</td>
</tr>
</tbody>
</table>
<p>The phenotype information is <a href="https://github.com/liangyy/phenotype-epistasis/blob/master/external-data/ncomms11122-s1-table-s1.pdf">here</a> (Table S1)</p>
</div>
<div id="personality" class="section level2">
<h2><span class="header-section-number">1.3</span> Personality</h2>
<ul>
<li><a href="https://www.nature.com/articles/ng.3736">Paper-complete</a></li>
<li><a href="http://www.tweelingenregister.org/fileadmin/user_upload/publicaties/verslaggeving/NTR-publicaties_2012/Moor_MP_2012.pdf">Paper-summary statistic available</a></li>
<li><a href="http://www.tweelingenregister.org/GPC/">Data</a></li>
<li>Meta-data</li>
</ul>
<table style="width:100%;">
<colgroup>
<col width="5%" />
<col width="14%" />
<col width="16%" />
<col width="14%" />
<col width="18%" />
<col width="20%" />
<col width="10%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">id</th>
<th align="center">phenotype</th>
<th align="center">num.of.traits</th>
<th align="center">sample.size</th>
<th align="center">num.of.variants</th>
<th align="center">heritability.mean</th>
<th align="center">ancestry</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">3</td>
<td align="center">personality</td>
<td align="center">5</td>
<td align="center">17375</td>
<td align="center">NA</td>
<td align="center">1%-12%</td>
<td align="center">european</td>
</tr>
</tbody>
</table>
<p>The phenotype information is <a href="https://github.com/liangyy/phenotype-epistasis/blob/master/external-data/ng.3736-S1-table-s2.pdf">here</a> (Table S2)</p>
<p>Keep in mind that the sample size is 17375 (63661 for neuroticism and extraversion) even if it is explicitly told in the readme file).</p>
</div>
<div id="summary" class="section level2">
<h2><span class="header-section-number">1.4</span> Summary</h2>
<table>
<colgroup>
<col width="4%" />
<col width="23%" />
<col width="14%" />
<col width="12%" />
<col width="16%" />
<col width="18%" />
<col width="9%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">id</th>
<th align="center">phenotype</th>
<th align="center">num.of.traits</th>
<th align="center">sample.size</th>
<th align="center">num.of.variants</th>
<th align="center">heritability.mean</th>
<th align="center">ancestry</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">1</td>
<td align="center">blood cell traits</td>
<td align="center">36</td>
<td align="center">173480</td>
<td align="center">29500000</td>
<td align="center">10%-30%</td>
<td align="center">european</td>
</tr>
<tr class="even">
<td align="center">2</td>
<td align="center">circulating metabolites</td>
<td align="center">123</td>
<td align="center">24925</td>
<td align="center">3.9e+07</td>
<td align="center">NA</td>
<td align="center">european</td>
</tr>
<tr class="odd">
<td align="center">3</td>
<td align="center">personality</td>
<td align="center">5</td>
<td align="center">17375</td>
<td align="center">NA</td>
<td align="center">1%-12%</td>
<td align="center">european</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="disease-phenotypes" class="section level1">
<h1><span class="header-section-number">2</span> Disease Phenotypes</h1>
<p>The individual level data is needed here since we need to calculate the predicted intermediate trait from it.</p>
<div id="wtccc" class="section level2">
<h2><span class="header-section-number">2.1</span> WTCCC</h2>
<p><a href="https://www.nature.com/articles/nature05911">WTCCC (2009)</a> has 7 complex diseases:</p>
<ol style="list-style-type: decimal">
<li>bipolar disorder (BD)</li>
<li>coronary artery disease (CAD)</li>
<li>Crohn’s disease (CD)</li>
<li>hypertension (HT)</li>
<li>rheumatoid arthritis (RA)</li>
<li>type 1 diabetes (T1D)</li>
<li>type 2 diabetes (T2D)</li>
</ol>
<p>2000 cases each and 3000 shared controls (european ancestry). Controls contains two groups, 1500 from 1958 British Birth Cohort (58C) and 1500 from blood donors.</p>
<p>Posterior mean genotype is available from Nick in BIMBAM format.</p>
</div>
<div id="autism" class="section level2">
<h2><span class="header-section-number">2.2</span> Autism</h2>
<p>TODO</p>
</div>
</div>
<div id="gwas-for-prs-validation" class="section level1">
<h1><span class="header-section-number">3</span> GWAS for PRS Validation</h1>
<p>Here, we focus on WTCCC phenotypes.</p>
<div id="type-2-diabetes" class="section level2">
<h2><span class="header-section-number">3.1</span> Type 2 Diabetes</h2>
<ul>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/MorrisAP_22885922_GCST005047">Data, DIAGRAMv3.2016MAY31</a></li>
<li><a href="https://www-nature-com.proxy.uchicago.edu/articles/ng.2383">Paper, DIAGRAMv3</a></li>
<li><strong>ATTENTION: WTCCC is included</strong></li>
<li>Meta-data</li>
</ul>
<table>
<colgroup>
<col width="4%" />
<col width="11%" />
<col width="14%" />
<col width="13%" />
<col width="16%" />
<col width="18%" />
<col width="20%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">id</th>
<th align="center">phenotype</th>
<th align="center">num.of.traits</th>
<th align="center">sample.size</th>
<th align="center">num.of.variants</th>
<th align="center">heritability.mean</th>
<th align="center">ancestry</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">1</td>
<td align="center">t2d</td>
<td align="center">1</td>
<td align="center">150000</td>
<td align="center">127903</td>
<td align="center">30%-70%</td>
<td align="center">european + Pakistani</td>
</tr>
</tbody>
</table>
<p>The sample information is <a href="https://github.com/liangyy/phenotype-epistasis/blob/master/external-data/ng.2383-S1-table-s1.pdf">here</a> (Table S1)</p>
</div>
<div id="crohns-disease" class="section level2">
<h2><span class="header-section-number">3.2</span> Crohn’s Disease</h2>
<ul>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/deLangeKM_28067908_GCST004132">Data</a></li>
<li><a href="https://www.nature.com/articles/ng.3760">Paper, Crohn’s disease</a></li>
<li>Meta-data</li>
</ul>
<table style="width:100%;">
<colgroup>
<col width="4%" />
<col width="17%" />
<col width="15%" />
<col width="13%" />
<col width="17%" />
<col width="19%" />
<col width="9%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">id</th>
<th align="center">phenotype</th>
<th align="center">num.of.traits</th>
<th align="center">sample.size</th>
<th align="center">num.of.variants</th>
<th align="center">heritability.mean</th>
<th align="center">ancestry</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">2</td>
<td align="center">crohn’s disease</td>
<td align="center">1</td>
<td align="center">12000</td>
<td align="center">9500000</td>
<td align="center">NA</td>
<td align="center">european</td>
</tr>
</tbody>
</table>
<p>The sample information is as follow (Table S1).</p>
<pre class="r"><code>data &lt;- read.csv(&#39;../external-data/ng.3760-S2-table-s1.csv&#39;)
kable(data, &#39;html&#39;) %&gt;%
  kable_styling() %&gt;%
  scroll_box(width = &#39;1000px&#39;, height = &#39;500px&#39;)</code></pre>
<div style="border: 1px solid #ddd; padding: 5px; overflow-y: scroll; height:500px; overflow-x: scroll; width:1000px; ">
<table class="table" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
Study
</th>
<th style="text-align:left;">
Data
</th>
<th style="text-align:left;">
Crohn.s.disease..CD.
</th>
<th style="text-align:left;">
X
</th>
<th style="text-align:left;">
X.1
</th>
<th style="text-align:left;">
Ulcerative.colitis..UC.
</th>
<th style="text-align:left;">
X.2
</th>
<th style="text-align:left;">
X.3
</th>
<th style="text-align:left;">
Inflammatory.bowel.disease..IBD.
</th>
<th style="text-align:left;">
X.4
</th>
<th style="text-align:left;">
X.5
</th>
<th style="text-align:left;">
Controls
</th>
<th style="text-align:left;">
X.6
</th>
<th style="text-align:left;">
X.7
</th>
<th style="text-align:left;">
Grand.Total
</th>
<th style="text-align:left;">
X.8
</th>
<th style="text-align:left;">
X.9
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Total
</td>
<td style="text-align:left;">
QC+
</td>
<td style="text-align:left;">
Non-overlap
</td>
<td style="text-align:left;">
Total
</td>
<td style="text-align:left;">
QC+
</td>
<td style="text-align:left;">
Non-overlap
</td>
<td style="text-align:left;">
Total
</td>
<td style="text-align:left;">
QC+
</td>
<td style="text-align:left;">
Non-overlap
</td>
<td style="text-align:left;">
Total
</td>
<td style="text-align:left;">
QC+
</td>
<td style="text-align:left;">
Non-overlap
</td>
<td style="text-align:left;">
Total
</td>
<td style="text-align:left;">
QC+
</td>
<td style="text-align:left;">
Non-overlap
</td>
</tr>
<tr>
<td style="text-align:left;">
UK low coverage whole genome sequencing IBD study
</td>
<td style="text-align:left;">
Samples
</td>
<td style="text-align:left;">
2,697
</td>
<td style="text-align:left;">
2513
</td>
<td style="text-align:left;">
1974
</td>
<td style="text-align:left;">
1817
</td>
<td style="text-align:left;">
1767
</td>
<td style="text-align:left;">
1326
</td>
<td style="text-align:left;">
4514
</td>
<td style="text-align:left;">
4280
</td>
<td style="text-align:left;">
3300
</td>
<td style="text-align:left;">
3910
</td>
<td style="text-align:left;">
3652
</td>
<td style="text-align:left;">
3650
</td>
<td style="text-align:left;">
8424
</td>
<td style="text-align:left;">
7932
</td>
<td style="text-align:left;">
6950
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Sites
</td>
<td style="text-align:left;">
95,140,340
</td>
<td style="text-align:left;">
13,272,215
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
95,140,340
</td>
<td style="text-align:left;">
13,219,710
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
95,140,340
</td>
<td style="text-align:left;">
13,211,989
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
95,140,340
</td>
<td style="text-align:left;">
13,211,989
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
95,140,340
</td>
<td style="text-align:left;">
13,211,989
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
UK HumanCoreExome genotyped IBD study
</td>
<td style="text-align:left;">
Samples
</td>
<td style="text-align:left;">
5695
</td>
<td style="text-align:left;">
4474
</td>
<td style="text-align:left;">
4264
</td>
<td style="text-align:left;">
5299
</td>
<td style="text-align:left;">
4173
</td>
<td style="text-align:left;">
4072
</td>
<td style="text-align:left;">
11758
</td>
<td style="text-align:left;">
9239
</td>
<td style="text-align:left;">
8860
</td>
<td style="text-align:left;">
10484
</td>
<td style="text-align:left;">
9500
</td>
<td style="text-align:left;">
9495
</td>
<td style="text-align:left;">
22242
</td>
<td style="text-align:left;">
18739
</td>
<td style="text-align:left;">
18355
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Genotyped sites
</td>
<td style="text-align:left;">
535434
</td>
<td style="text-align:left;">
296203
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
535434
</td>
<td style="text-align:left;">
296203
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
535434
</td>
<td style="text-align:left;">
296203
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
535434
</td>
<td style="text-align:left;">
296203
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
535434
</td>
<td style="text-align:left;">
296203
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Imputed sites
</td>
<td style="text-align:left;">
109,485,955
</td>
<td style="text-align:left;">
18,956,986
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
109,485,955
</td>
<td style="text-align:left;">
18,940,631
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
109,485,955
</td>
<td style="text-align:left;">
18,954,471
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
109,485,955
</td>
<td style="text-align:left;">
18,954,471
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
109,485,955
</td>
<td style="text-align:left;">
18,954,471
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
IIBDGC genotyped CD study (summary statistics)
</td>
<td style="text-align:left;">
Samples
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
5956
</td>
<td style="text-align:left;">
5956
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
14927
</td>
<td style="text-align:left;">
14927
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
20883
</td>
<td style="text-align:left;">
20883
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Sites
</td>
<td style="text-align:left;">
12,276,506
</td>
<td style="text-align:left;">
12,036,635
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
12,276,506
</td>
<td style="text-align:left;">
12,036,635
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
12,276,506
</td>
<td style="text-align:left;">
12,036,635
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
IIBDGC genotyped UC study (summary statistics)
</td>
<td style="text-align:left;">
Samples
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
6968
</td>
<td style="text-align:left;">
6968
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
20464
</td>
<td style="text-align:left;">
20464
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
27432
</td>
<td style="text-align:left;">
27432
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Sites
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
12,255,263
</td>
<td style="text-align:left;">
12,055,356
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
12,255,263
</td>
<td style="text-align:left;">
12,055,356
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
12,255,263
</td>
<td style="text-align:left;">
12,055,356
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
IIBDGC genotyped IBD study (summary statistics)
</td>
<td style="text-align:left;">
Samples
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
12882
</td>
<td style="text-align:left;">
12882
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
21770
</td>
<td style="text-align:left;">
21770
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
34652
</td>
<td style="text-align:left;">
34652
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Sites
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
12,716,150
</td>
<td style="text-align:left;">
12,484,467
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
12,716,150
</td>
<td style="text-align:left;">
12,484,467
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
12,716,150
</td>
<td style="text-align:left;">
12,484,467
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Combined meta-analysis (CD | UC | IBD)
</td>
<td style="text-align:left;">
Samples
</td>
<td style="text-align:left;">
12194
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
12366
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
25042
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
28072 | 33609 | 34915
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
40266 | 45975 | 59957
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Sites
</td>
<td style="text-align:left;">
20,765,806
</td>
<td style="text-align:left;">
9,560,910
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
20,764,549
</td>
<td style="text-align:left;">
9,578,670
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
20,936,273
</td>
<td style="text-align:left;">
9,725,440
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
9,560,910 | 9,578,670 | 9,725,440
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
9,560,910 | 9,578,670 | 9,725,440
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="summary-1" class="section level2">
<h2><span class="header-section-number">3.3</span> Summary</h2>
<table style="width:100%;">
<colgroup>
<col width="4%" />
<col width="15%" />
<col width="14%" />
<col width="12%" />
<col width="15%" />
<col width="17%" />
<col width="19%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">id</th>
<th align="center">phenotype</th>
<th align="center">num.of.traits</th>
<th align="center">sample.size</th>
<th align="center">num.of.variants</th>
<th align="center">heritability.mean</th>
<th align="center">ancestry</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">1</td>
<td align="center">t2d</td>
<td align="center">1</td>
<td align="center">150000</td>
<td align="center">127903</td>
<td align="center">30%-70%</td>
<td align="center">european + Pakistani</td>
</tr>
<tr class="even">
<td align="center">2</td>
<td align="center">crohn’s disease</td>
<td align="center">1</td>
<td align="center">12000</td>
<td align="center">9500000</td>
<td align="center">NA</td>
<td align="center">european</td>
</tr>
</tbody>
</table>
</div>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
