<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Implementation</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-1.1/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-1.1/highlight.js"></script>
<link href="site_libs/font-awesome-4.5.0/css/font-awesome.min.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Phenotypic Genetic Background Effect</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="data.html">Data</a>
</li>
<li>
  <a href="computation.html">Computation</a>
</li>
<li>
  <a href="implementation.html">Implementation</a>
</li>
<li>
  <a href="results.html">Results</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/liangyy/phenotype-epistasis">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Implementation</h1>
<h4 class="date"><em>28 February, 2018</em></h4>

</div>


<p><span class="math display">\[
\newcommand\logit{\text{logit}}  
\]</span></p>
<div id="step-1-beta_i-x_t-rightarrow-haty_i" class="section level1">
<h1><span class="header-section-number">1</span> Step 1: <span class="math inline">\((\beta_I, X_T) \rightarrow \hat{Y}_I\)</span></h1>
<p>Current available <span class="math inline">\(f: (\beta_I, X_T) \rightarrow \hat{Y}_I\)</span> are <code>PRSice</code> and <code>LDpred</code> (see <a href="computation.html#1_computing_intermediate_phenotype">here</a>). The available <span class="math inline">\(X_T\)</span>, WTCCC genotype, is in BIMBAM format.</p>
<div id="prepare-x_t" class="section level2">
<h2><span class="header-section-number">1.1</span> Prepare <span class="math inline">\(X_T\)</span></h2>
<p>Both <code>PRSice</code> and <code>LDpred</code> take plink binary, so <span class="math inline">\(X_T\)</span> should be converted to plink binary format. It can be obtained by <a href="https://vcru.wisc.edu/simonlab/bioinformatics/programs/fcgene/fcgene-1.0.7.pdf"><code>fcgene</code> chapter 9</a>.</p>
<blockquote>
<p>Side notes: It turns out there are several difficulties to use <code>fcgene</code>:</p>
<ol style="list-style-type: decimal">
<li><code>fcgene</code> can only take genotype distribution file instead of mean genotype file. One way to work around is to generate a “fake” genotype distribution file. Namely, genotype distribution is set to one-hot vector where only the genotype that is closest to the mean genotype is assigned to 1;</li>
<li><code>fcgene</code> flips the minor and major alleles. So, to work around, we need to flip the order of allele in genotype distribution file (label it as <code>genotype-dist-flip</code>). Note that only the order in genotype file matters and SNP info file does not;</li>
<li><code>fcgene</code> takes an information file for SNPs with MAF, which is not available but since it is not included in the output we can add a fake MAF (say 0.5).</li>
</ol>
</blockquote>
<pre><code># Pseudo-code for this task

## comment 1 &amp; 2
$ mean2dist_genotype --input test.genotype-mean.txt --output test.genotype-dist-flip.txt

## comment 3
$ gen_snpinfo --input test.pos --output test.bimbam.snpinfo

## run fcgene
$ fcgene --wgd test.genotype-dist-flip.txt --pos test.bimbam.snpinfo --oformat plink-bed --out test.bplink

## outcome files
$ ls
test.bplink.bed test.bplink.bim test.bplink.fam

## *.fam output by fcgene needs post-processing to remove extra space at the end of each column</code></pre>
</div>
<div id="prepare-beta_i" class="section level2">
<h2><span class="header-section-number">1.2</span> Prepare <span class="math inline">\(\beta_I\)</span></h2>
<p>The following format would work well for both methods.</p>
<pre><code>chr     pos     ref     alt     reffrq  info    rs       pval    effalt
chr1  1020428 C T 0.85083 0.98732 rs6687776 0.0587  -0.0100048507289348
chr1  1020496 G A 0.85073 0.98751 rs6678318 0.1287  -0.00826075392985992</code></pre>
</div>
<div id="pipeline" class="section level2">
<h2><span class="header-section-number">1.3</span> Pipeline</h2>
<p>Only the approach using Ldpred has been implemented (at <a href="https://github.com/liangyy/phenotype-epistasis/tree/master/code/step1"><code>code/step1/</code></a>)</p>
<!-- ## Prepare fake $Y_I$ -->
<!-- `PRSice` (but not `LDpred`) needs $Y_t$ to optimize prediction accuracy by tuning hyper-parameters. As the true phenotype is not available, here we generate a "fake" $Y_t$ instead which won't affect the computation. -->
</div>
<div id="evaluation" class="section level2">
<h2><span class="header-section-number">1.4</span> Evaluation</h2>
<p>Leave it for future.</p>
</div>
</div>
<div id="step-2-y_t-sim-x_t-cdot-haty_i" class="section level1">
<h1><span class="header-section-number">2</span> Step 2: <span class="math inline">\(Y_T \sim X_T \cdot \hat{Y}_I\)</span></h1>
Current we test the following two types of interactions
<span class="math display">\[\begin{align*}
  \logit(\Pr(Y_T|X_T, Y_I)) &amp;= \alpha + \gamma Y_I + \beta_1 X_T + \beta_2 X_T Y_I\\
  \logit(\Pr(Y_T|X_T, Y_I)) &amp;= \alpha + \gamma Y_I + \beta_{11} X_T + \beta_{12} X_T Y_I + \beta_{21} X_T + \beta_{22} X_T Y_I 
\end{align*}\]</span>
<p>, where the first is additive model and the second is dominant/recessive model. See detail <a href="computation.html#model">here</a>.</p>
<div id="combine-x_t-and-y_t" class="section level2">
<h2><span class="header-section-number">2.1</span> Combine <span class="math inline">\(X_T\)</span> and <span class="math inline">\(Y_T\)</span></h2>
<pre><code># change the last column to phenotype (1 unaffected, 2 affected)
$ bash add_phenotype.sh -p 1 -i data1.fam -o data1_phenotyped.fam
$ bash add_phenotype.sh -p 2 -i data2.fam -o data2_phenotyped.fam

# rename *.bim and *.bed files accordingly
$ mv data1.bim data1_phenotyped.bim.fam; mv data1.bed data1_phenotyped.bed
$ mv data2.bim data2_phenotyped.bim.fam; mv data2.bed data2_phenotyped.bed

# merge two data sets
$ plink --bfile data1_phenotyped --bmerge data2_phenotyped --make-bed --out merged_data

# QC
$ plink --bfile merged_data --make-bed --mind 0.05 --maf 0.01 --geno 0.05 --hwe 0.05 --out merged_data_QC</code></pre>
</div>
<div id="get-pcs-of-x_t" class="section level2">
<h2><span class="header-section-number">2.2</span> Get PCs of <span class="math inline">\(X_T\)</span></h2>
<pre><code>Rscript getQC.R --geno merged_data_QC -nPCs 2 --output PCs.merged_data_QC.txt.gz</code></pre>
</div>
<div id="determine-the-direction-of-effect" class="section level2">
<h2><span class="header-section-number">2.3</span> Determine the direction of effect</h2>
<div id="direction-of-x_t" class="section level3">
<h3><span class="header-section-number">2.3.1</span> Direction of <span class="math inline">\(X_T\)</span></h3>
<pre><code>$ plink --bfile merged_data_QC --logistic --out log_assoc_merged_data_QC --covar &lt;(zcat &lt; PCs.merged_data_QC.txt.gz)
$ Rscript extractDirection.R --input log_assoc.merged_data_QC.logistic --output direction_XT.merge_data_QC.txt.gz</code></pre>
</div>
<div id="direction-of-y_i" class="section level3">
<h3><span class="header-section-number">2.3.2</span> Direction of <span class="math inline">\(Y_I\)</span></h3>
<pre><code># predict Y_I based on merged_data_QC
$ python ldpred/validate.py SOMETHING --input merged_data_QC --out merged_data_QC_YI  # ignore the prediction accuracy since the phenotype is wrong

# determine the direction
$ Rscript computeDirection.R --input merged_data_QC_YI.txt.gz --output directon_YI.merged_data_QC.txt.gz</code></pre>
</div>
</div>
<div id="perform-interaction-test" class="section level2">
<h2><span class="header-section-number">2.4</span> Perform interaction test</h2>
<pre><code># merge covariates
bash merge_two_covar.sh -a YI.merged_data_QC.txt.gz -b PCs.merged_data_QC.txt.gz -o covars.merged_data_QC.txt.gz

# additive model
$ plink --bfile merged_data_QC --logistic --covar &lt;(zcat &lt; YI.merged_data_QC.txt.gz) --interaction --out interaction_add_YI.merged_data_QC

# dominant/recessive model
## note that the dominant/recessive is respective to reference allele but not the risk one
## in the end, the dominant and recessive will be reassigned according to direction files
$ plink --bfile merged_data_QC --logistic --covar &lt;(zcat &lt; YI.merged_data_QC.txt.gz) --interaction --dominant --out interaction_dom_YI.merged_data_QC
$ plink --bfile merged_data_QC --logistic --covar &lt;(zcat &lt; YI.merged_data_QC.txt.gz) --interaction --recessive --out interaction_rec_YI.merged_data_QC</code></pre>
</div>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
