<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Results</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-4.5.0/css/font-awesome.min.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Phenotypic Genetic Background Effect</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="data.html">Data</a>
</li>
<li>
  <a href="computation.html">Computation</a>
</li>
<li>
  <a href="implementation.html">Implementation</a>
</li>
<li>
  <a href="results.html">Results</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/liangyy/phenotype-epistasis">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Results</h1>
<h4 class="date"><em>28 February, 2018</em></h4>

</div>


<div id="summary" class="section level1">
<h1><span class="header-section-number">1</span> Summary</h1>
<p>This page presents the results of analysis that have been done. Every section is one disease (phenotype of interest) and each subsection is the analysis for a particular intermediate trait.</p>
<p>Every subsection presents the following information.</p>
<ul>
<li>Comparison between raw <span class="math inline">\(\beta\)</span> and posterior mean of <span class="math inline">\(\beta\)</span> for computing PRS</li>
<li>Marginal association:
<ul>
<li><span class="math inline">\(Y \sim I\)</span></li>
<li><span class="math inline">\(Y \sim G_i\)</span></li>
<li><span class="math inline">\(Y \sim I + G_i\)</span></li>
</ul></li>
<li>Interaction test results based on functional subsetting</li>
<li>Follow up comment on the results if any</li>
</ul>
</div>
<div id="coronary-heart-disease" class="section level1">
<h1><span class="header-section-number">2</span> Coronary heart disease</h1>
<div id="hdl" class="section level2">
<h2><span class="header-section-number">2.1</span> HDL</h2>
<pre class="r"><code>config.hdl &lt;- &#39;../code/step2/config.wtccc-cad-hdl.yaml&#39;
doAll(config.hdl)</code></pre>
<p>The threhold used for extracting functionally consistent variant is 0.001 <img src="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-direction_XT_YI/MAGNETIC_HDL.C__wtccc_cad__wtccc_ctrl/direction_yi.png" /></p>
<ol style="list-style-type: decimal">
<li>Comparison of Beta <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step1/output-run_step1/MAGNETIC_HDL.C__wtccc_cad__wtccc_ctrl/prs_report.html">here</a></li>
<li>Marginal directions <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-direction_XT_YI/MAGNETIC_HDL.C__wtccc_cad__wtccc_ctrl/direction_report.html">here</a></li>
<li>Interaction test results are <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-run_step2/MAGNETIC_HDL.C__wtccc_cad__wtccc_ctrl/inter_report.func_subset.html">here</a></li>
</ol>
</div>
<div id="ldl" class="section level2">
<h2><span class="header-section-number">2.2</span> LDL</h2>
<pre class="r"><code>config.ldl &lt;- &#39;../code/step2/config.wtccc-cad-ldl.yaml&#39;
doAll(config.ldl)</code></pre>
<p>The threhold used for extracting functionally consistent variant is 0.01 <img src="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-direction_XT_YI/MAGNETIC_LDL.C__wtccc_cad__wtccc_ctrl/direction_yi.png" /></p>
<ol style="list-style-type: decimal">
<li>Comparison of Beta <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step1/output-run_step1/MAGNETIC_LDL.C__wtccc_cad__wtccc_ctrl/prs_report.html">here</a></li>
<li>Marginal directions <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-direction_XT_YI/MAGNETIC_LDL.C__wtccc_cad__wtccc_ctrl/direction_report.html">here</a></li>
<li>Interaction test results are <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-run_step2/MAGNETIC_LDL.C__wtccc_cad__wtccc_ctrl/inter_report.func_subset.html">here</a></li>
</ol>
<p>This <a href="http://circres.ahajournals.org/content/118/1/83">paper</a> shows the strong genetic correlation between coronary heart disease and LDL and conditional FDR (on LDL; namely incorporating LDL p-value into FDR calculation) of CHD help to identify more signal out of GWAS follow-up study. Also, LDL has been shown to be causal factor of CHD, <a href="http://www.nejm.org.proxy.uchicago.edu/doi/full/10.1056/NEJMoa054013">paper</a>.</p>
<p><a href="https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=13465"><em>rs13465</em></a> locates in the intronic region of ILF3. Its protective effect is masked by high LDL level.</p>
<pre class="r"><code>drawGraph(&#39;-&#39;, &#39;-&#39;, &#39;+&#39;, &#39;+&#39;, &#39;cad-ldl&#39;) #</code></pre>
<div class="figure">
<img src="intergraph.cad-ldl.png" />

</div>
<p>Interestingly, this <a href="https://www.spandidos-publications.com/mmr/4/3/511">paper</a> reported that ILF3 (<em>rs2569512</em> was used to tag the gene) affects myocardial infarction risk only when LDL level is low (when LDL level is high, there is no association signal).</p>
</div>
<div id="serum-triglycerides-concentration" class="section level2">
<h2><span class="header-section-number">2.3</span> Serum triglycerides concentration</h2>
<pre class="r"><code>config.tg &lt;- &#39;../code/step2/config.wtccc-cad-tg.yaml&#39;
doAll(config.tg)</code></pre>
<p>The threhold used for extracting functionally consistent variant is 0.01 <img src="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-direction_XT_YI/MAGNETIC_Serum.TG__wtccc_cad__wtccc_ctrl/direction_yi.png" /></p>
<ol style="list-style-type: decimal">
<li>Comparison of Beta <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step1/output-run_step1/MAGNETIC_Serum.TG__wtccc_cad__wtccc_ctrl/prs_report.html">here</a></li>
<li>Marginal directions <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-direction_XT_YI/MAGNETIC_Serum.TG__wtccc_cad__wtccc_ctrl/direction_report.html">here</a></li>
<li>Interaction test results are <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-run_step2/MAGNETIC_Serum.TG__wtccc_cad__wtccc_ctrl/inter_report.func_subset.html">here</a></li>
</ol>
<p>This paper proposed that triglycerides level is also a causal factor of CHD, <a href="https://www-nature-com.proxy.uchicago.edu/articles/ng.2795">paper</a>.</p>
<p><a href="https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=4541125"><em>rs4541125</em></a> shows most significant interaction (positive: i.e. as <span class="math inline">\(I\)</span> increases, the effect size of <span class="math inline">\(G\)</span> increases). In other word, the protective effect of <span class="math inline">\(G\)</span> goes away when <span class="math inline">\(I\)</span> is high. It locates in the intronic region of PITPNM3, which is reported to be under-presented in the proteomics of Acute Ischemic Strokes patients (<a href="https://pubs.acs.org/doi/full/10.1021/pr501324n?src=recsys">table 3 of the paper</a>). Note that the triglycerides level is high in cases versus controls.</p>
<pre class="r"><code>drawGraph(&#39;-&#39;, &#39;-&#39;, &#39;+&#39;, &#39;+&#39;, &#39;cad-tg&#39;)  # gy, gi, yi</code></pre>
<div class="figure">
<img src="intergraph.cad-tg.png" />

</div>
</div>
<div id="leptin" class="section level2">
<h2><span class="header-section-number">2.4</span> Leptin</h2>
<pre class="r"><code>config.leptin &lt;- &#39;../code/step2/config.wtccc-cad-leptin.yaml&#39;
doAll(config.leptin)</code></pre>
<p>The threhold used for extracting functionally consistent variant is 0.01 <img src="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-direction_XT_YI/LeptinBMI__wtccc_cad__wtccc_ctrl/direction_yi.png" /></p>
<ol style="list-style-type: decimal">
<li>Comparison of Beta <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step1/output-run_step1/LeptinBMI__wtccc_cad__wtccc_ctrl/prs_report.html">here</a></li>
<li>Marginal directions <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-direction_XT_YI/LeptinBMI__wtccc_cad__wtccc_ctrl/direction_report.html">here</a></li>
<li>Interaction test results are <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-run_step2/LeptinBMI__wtccc_cad__wtccc_ctrl/inter_report.func_subset.html">here</a></li>
</ol>
</div>
</div>
<div id="crohns-disease" class="section level1">
<h1><span class="header-section-number">3</span> Crohn’s disease</h1>
<div id="c-reactive-protein" class="section level2">
<h2><span class="header-section-number">3.1</span> C-reactive protein</h2>
<p>It turns out that the GWAS signal goes away if limiting to loci which occur in WTCCC genotype data.</p>
</div>
<div id="insulin-like-growth-factor" class="section level2">
<h2><span class="header-section-number">3.2</span> Insulin-like growth factor</h2>
<pre class="r"><code>config.igf1 &lt;- &#39;../code/step2/config.wtccc-crohn-igf1.yaml&#39;
doAll(config.igf1)</code></pre>
<p>The threhold used for extracting functionally consistent variant is 0.01 <img src="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-direction_XT_YI/Prins_igf1__wtccc_crohn__wtccc_ctrl/direction_yi.png" /></p>
<ol style="list-style-type: decimal">
<li>Comparison of Beta <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step1/output-run_step1/Prins_igf1__wtccc_crohn__wtccc_ctrl/prs_report.html">here</a></li>
<li>Marginal directions <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-direction_XT_YI/Prins_igf1__wtccc_crohn__wtccc_ctrl/direction_report.html">here</a></li>
<li>Interaction test results are <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-run_step2/Prins_igf1__wtccc_crohn__wtccc_ctrl/inter_report.func_subset.html">here</a></li>
</ol>
</div>
<div id="white-blood-cell-count" class="section level2">
<h2><span class="header-section-number">3.3</span> White blood cell count</h2>
<pre class="r"><code>config.wbc &lt;- &#39;../code/step2/config.wtccc-crohn-wbc.yaml&#39;
doAll(config.wbc)</code></pre>
<p>The threhold used for extracting functionally consistent variant is 0.001 <img src="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-direction_XT_YI/UKBB_wbc__wtccc_crohn__wtccc_ctrl/direction_yi.png" /></p>
<ol style="list-style-type: decimal">
<li>Comparison of Beta <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step1/output-run_step1/UKBB_wbc__wtccc_crohn__wtccc_ctrl/prs_report.html">here</a></li>
<li>Marginal directions <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-direction_XT_YI/UKBB_wbc__wtccc_crohn__wtccc_ctrl/direction_report.html">here</a></li>
<li>Interaction test results are <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-run_step2/UKBB_wbc__wtccc_crohn__wtccc_ctrl/inter_report.func_subset.html">here</a></li>
</ol>
<p><a href="https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=7775228"><em>rs7775228</em></a> is a tag SNP for HLA DQ2.2 haplotype <a href="https://www.snpedia.com/index.php/Rs7775228">link</a>.</p>
<p>Interestingly, when adding a sparse prior on the effect size, two SNPs (<a href="https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=4379550"><em>rs4379550</em></a> and <a href="https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=rs11145756"><em>rs11145756</em></a> are both located in the intronic region of SEC16A. And their interaction terms is negative, indicating that the SNP and intermediate trait act in redundant manner.</p>
<pre class="r"><code>drawGraph(&#39;+&#39;, &#39;+&#39;, &#39;+&#39;, &#39;-&#39;, &#39;crohn-wbc&#39;)  # gy, gi, yi</code></pre>
<div class="figure">
<img src="intergraph.crohn-wbc.png" />

</div>
</div>
</div>
<div id="type-ii-diabetes" class="section level1">
<h1><span class="header-section-number">4</span> Type II diabetes</h1>
<div id="fast-glucose" class="section level2">
<h2><span class="header-section-number">4.1</span> Fast glucose</h2>
<pre class="r"><code>config.fastglu &lt;- &#39;../code/step2/config.wtccc-t2d-fastglu.yaml&#39;
doAll(config.fastglu)</code></pre>
<p>The threhold used for extracting functionally consistent variant is 0.01 <img src="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-direction_XT_YI/fastGlu__wtccc_t2d__wtccc_ctrl/direction_yi.png" /></p>
<ol style="list-style-type: decimal">
<li>Comparison of Beta <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step1/output-run_step1/fastGlu__wtccc_t2d__wtccc_ctrl/prs_report.html">here</a></li>
<li>Marginal directions <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-direction_XT_YI/fastGlu__wtccc_t2d__wtccc_ctrl/direction_report.html">here</a></li>
<li>Interaction test results are <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-run_step2/fastGlu__wtccc_t2d__wtccc_ctrl/inter_report.func_subset.html">here</a></li>
</ol>
<p><a href="https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=10509645"><em>rs10509645</em></a> locates in the intronic region of <a href="http://www.uniprot.org/uniprot/P14735">IDE</a>.</p>
<p><a href="https://www.snpedia.com/index.php/Rs5015480"><em>rs5015480</em></a> (gene: <a href="http://www.uniprot.org/uniprot/Q03014">HHEX</a>).</p>
<p>For <a href="https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=275775"><em>rs1243150</em></a> and <a href="https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=275774"><em>rs1243149</em></a>, the gene is unknown.</p>
<p><a href="https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=rs7923866"><em>rs7923866</em></a> is also in the vicinity of the HHEX (homeobox, hematopoietically expressed) and IDE (insulin-degrading enzyme) <a href="https://www.nature.com/articles/nature05911">link</a>.</p>
<p><a href="https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=760233"><em>rs760233</em></a> locates in the intronic region of <a href="https://www.uniprot.org/uniprot/Q6EMB2">TTLL5</a>, which is related to retinal development and has not been reported to be associated with T2D.</p>
<!--

[*10811661*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=10811661) is strongly associated with T2D [link](https://www.snpedia.com/index.php/Rs10811661) which is located at upstream of CDKN2A/B. [This paper](http://diabetes.diabetesjournals.org/content/early/2018/02/05/db17-1055) proposed that *CDKN2A/B locus SNPs may impact T2D risk by modulating islet gene expression and beta
cell proliferation*.

[*rs2383208*](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2383208)  also locates near CDKN2A/2B [link](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0026953). -->
<p>Overall, all the SNPs mentioned above have the same direction. Namely the protective effect is masked if the fast glucose background level is high.</p>
<pre class="r"><code>drawGraph(&#39;-&#39;, &#39;-&#39;, &#39;+&#39;, &#39;+&#39;, &#39;t2d-fastglu&#39;)  # gy, gi, yi</code></pre>
<div class="figure">
<img src="intergraph.t2d-fastglu.png" />

</div>
</div>
<div id="fast-insulin" class="section level2">
<h2><span class="header-section-number">4.2</span> Fast insulin</h2>
<pre class="r"><code>config.fastinsulin &lt;- &#39;../code/step2/config.wtccc-t2d-fastinsulin.yaml&#39;
doAll(config.fastinsulin)</code></pre>
<p>The threhold used for extracting functionally consistent variant is 0.01 <img src="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-direction_XT_YI/fastInsulin__wtccc_t2d__wtccc_ctrl/direction_yi.png" /></p>
<ol style="list-style-type: decimal">
<li>Comparison of Beta <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step1/output-run_step1/fastInsulin__wtccc_t2d__wtccc_ctrl/prs_report.html">here</a></li>
<li>Marginal directions <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-direction_XT_YI/fastInsulin__wtccc_t2d__wtccc_ctrl/direction_report.html">here</a></li>
<li>Interaction test results are <a href="https://rawgit.com/liangyy/phenotype-epistasis/master/code/step2/output-run_step2/fastInsulin__wtccc_t2d__wtccc_ctrl/inter_report.func_subset.html">here</a></li>
</ol>
<p><a href="https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2155105"><em>rs2155105</em></a> locates in the intronic region of <a href="http://www.uniprot.org/uniprot/P57729">RAB38</a>. It is associated with albuminuria in T2D.</p>
<p>rs1996913, rs17606320, rs7544694, rs7543693, rs12173920, rs16895362,rs2049316, rs6864348 have almost no signal in <span class="math inline">\(I \sim G\)</span>.</p>
<p><a href="https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=3734262"><em>rs3734262</em></a> locates in the intronic region of <a href="http://www.uniprot.org/uniprot/Q96T88">UHRF1</a> binding protein 1 (<a href="http://www.uniprot.org/uniprot/Q6BDS2">UHRF1BP1</a>)</p>
<p><a href="https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=9368834"><em>rs9368834</em></a> locates in the intronic region of <a href="https://www.uniprot.org/uniprot/Q92625">ANKS1A</a>. It has been reported that this gene is associated with T2D in obese individuals.</p>
<p><a href="https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2139980"><em>rs2139980</em></a>. It direction is not the same as others.</p>
<p>All SNPs but rs2139980 mentioned has protective effect on T2D and when background fast insulin is high, the protective effects become smaller. For rs2139980, its protective effect becomes even stronger.</p>
<pre class="r"><code>drawGraph(&#39;-&#39;, &#39;-&#39;, &#39;+&#39;, &#39;+&#39;, &#39;t2d-fastinsulin&#39;)  # gy, gi, yi</code></pre>
<div class="figure">
<img src="intergraph.t2d-fastinsulin.png" />

</div>
</div>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
